From 97c51d8fe016148e9b09be07b1bf8b4b021a4880 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Tue, 27 Oct 2026 12:08:53 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh 757FAB7CED1814AE15B4836E5FB027474203454C.gpg from
keyservers/WKD: same certificate(s), with self-signatures our copy was
missing; the commands are in the spec comment.

Assisted-by: Claude Opus 5.5
---
 757FAB7CED1814AE15B4836E5FB027474203454C.gpg | 961 +++++++++++--------
 b43-fwcutter.spec                            |  10 +-
 2 files changed, 584 insertions(+), 387 deletions(-)

diff --git a/757FAB7CED1814AE15B4836E5FB027474203454C.gpg b/757FAB7CED1814AE15B4836E5FB027474203454C.gpg
index a985550..0c63042 100644
--- a/757FAB7CED1814AE15B4836E5FB027474203454C.gpg
+++ b/757FAB7CED1814AE15B4836E5FB027474203454C.gpg
@@ -1,6 +1,6 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
 
-mQINBFIF8KsBEACqe90N9wwjYw4QCPJ8HV47d4Vo8yUZ0gSgpWYIBocK9Zs0yqOJ
+xsFNBFIF8KsBEACqe90N9wwjYw4QCPJ8HV47d4Vo8yUZ0gSgpWYIBocK9Zs0yqOJ
 ek+bA2Z+0sI7CxO1NYfg7xJE6GpdUL9QFOK4cbJoBlZ0YuZlKFKvRiG+rsevY9FJ
 SZtKTwciW68pU2W5ixtW4g9FPq2LrqsrkGthNtSeL7uiK0SHtFGqz3k167aGfG8i
 l/VXkLRL8Vbl5/6mzE4FUwNcomS+dyOgaJHg2C4V/Lio0R4MjY8Lp6PVIPfKqa1d
@@ -11,7 +11,7 @@ HXoxn3s7ieA+4D5PjRB/8r+LMYOTxw78eEhJna6Ijvyew4FKWvvPgoFAOl9SNyOi
 5lTtwEzY1nxtYOZEBU5Bh/iFrzq6afwRmXUjsHnmb9MIgbSiep4IECwosBRRhvWu
 aRMgIOTbGiawt7/9/w2XUXgeMm3ZYotAm7ApawDSuFYVYJAATDTwHnYE15eK13Jh
 qeahCY4FMZXbc4WxD3anGj5ACAqdwK7nU/REU67eUGsxz/x3qrfEr5ol1QARAQAB
-tB1NYXggS2VsbGVybWFubiA8bWF4QGJsYXJnLmRlPokCOAQTAQIAIgUCUgXyaAIb
+zR1NYXggS2VsbGVybWFubiA8bWF4QGJsYXJnLmRlPsLBeAQTAQIAIgUCUgXyaAIb
 AwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AACgkQI26KWMbbRRItHxAAkAzF57av
 9FOLKOx1f/moQjUS7sWpYDDyAFLximo2CamFz3PupByKeZ/0g27j9YS7Jp7gbqo/
 Trp7cTY06lU5iaHHiaJxZlm1jK0MZuvzbAPQvRGj/kfaIiiEKq0ZwX0I1CcTaD8s
@@ -23,45 +23,45 @@ GublyBLmITCThRQZq5JZr7OI/zyrKfOYCjTL+yz6Yh8hqhJtNOpeLJRxfnjLURIN
 cTpd7fHEYBKG4FTn+p+81sGNJBTIh0avl0ScK7laygznis6lUFBI9cwfCXGVgbPx
 3xrEj9PSAogobWoIZvOPovrdvNvJht0VlHWoQ5P87w6jgbplzJAcsbSN2T8wPYfE
 DIP5UV59W7NruPQs8LYsAgMXTz0x18wiTWjPZNneZRfVZBQfe+RRtw6jCSyWR2kI
-sp39E/1Lj47+X20NGTk5Iom3xBydd8vftEG0IE1heCBLZWxsZXJtYW5uIDxtYXhA
-bXVzaWNwZC5vcmc+iQI4BBMBAgAiBQJSBfJgAhsDBgsJCAcDAgYVCAIJCgsEFgID
-AQIeAQIXgAAKCRAjbopYxttFEuXRD/4tlKQOLGb/lbYBlC+FOhnord47XMo/2F6J
-3C1LI22tZsGrMlnegFRZOJ39hBaZeK1hS3S8ho5Dw7sqdPw7+Bd9GLRK/ias+J1x
-pPQ0JiFe4SleVePHIFIPDRIzeOIVIKKeyt9cVDImru6VRV4faOHZ+CDbj/y8UUcm
-KVZww2jVK7W9JzVVyV0QM8FaJKZ2ih4h6L7uiYMv5uQsGNjR6wVjTIUTGEyM4R00
-nrA1LWYd+sfQzshH32TEm42yU/UB385PLyZ6Rgcau6DPqO6B5P1yH/oILXjI+ndQ
-sG7j3qVM6VELKr8PgZpzBM+0PI/hm8Fsi8ojMmvvzVfhjtd1CFWngvVwJKkGL82G
-EJkoBoUbTCpk0ZbdaYREvDh0aMGIIzaee9bGPuv0E/h9wYSKZk9Dnse2j8WdBENZ
-xhrBktgm71/J6lTJZ9PhYAmJOTvACpT0Kq/nGZ/mgq0gtzhrKErYSqFT7KwIUHIn
-dTjN2oFFU9bZNlZZFiRxN7GxAJYFHzAz8/7fM5R2qWxb/jZIL6IFWS4RrDDDWDuJ
-Zaw89kPhWcMA78CQ7xCpeNMUKErCll9Ihz2lUdQZfkoAt+UGvAvCeQkM7BqckV/n
-IVPbQR2kMQKUwdOQu5svIaWtoBxf96ryabPHhhPvFZYcQ0x6kcxO67DoLhWOyDfd
-u4jvNmAIZLQfTWF4IEtlbGxlcm1hbm4gPG1heEB4Y3NvYXIub3JnPokCOAQTAQIA
-IgUCUgXyUwIbAwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AACgkQI26KWMbbRRKa
-1g//VXBc5/lkUBmlpGTp7up0qecik9NiEpKVyQ5AHR1qi+VNBfL3YjlM/j6UY647
-P/t2qpLTsJ7MxrbX2Pigx0LnFtasrLgJ1knE+GZdqYW7XB1TiNBoA8YMGSeZBujz
-UpKsjcpU2SOhNDgmk+r7GH2lVgGU0Rrj6rU+e/wJj7l00Q/Vr9MtUH5MU1ogfM6c
-qhcj11pu680vVidd3A+VDTiQAL/0k4itp3cEtJk/bAJX0KksFZnvXeWtF38uK69P
-4JL0oGGW4lRZdwDIRuPvm1/z0lU7jns40TtbZA7v9/xH8F/eJNHkzX7zj6qaoq4F
-mJ2b2AXVINa+elIQWZ9hPdEEQGGOMWP13wfIaixIENREDnf8cQe/wMgbo+lDinFH
-4gmaUe20qM0hbFoQu3tFRVzDKb0+JqbVTTsBWvQANs8BSPVSBZP9M8fDwHVe7g0N
-msX4luM08GUVT/WsyMoCz0U4VlIHaV4Ep8cOvrt2+/xHlEN3fvPC0NgF0FJLI4My
-fHOTQ9PKHqxAz1WzyDsmYe1qkl5VY5rQ9U2IIIUHnhUAKy9KnFh8/q8RgrMZ2CJE
-M7oyAfo9gcYoSUlaz0w0zZK0OdRCdcpQDbfz9F8sqMJ6qlNGbnznSqqKo4ZUpkUW
-ufBbXZM99wN6KcsuEF+1WKpnz7dBgSMp3NjCespYrlC95pq0IE1heCBLZWxsZXJt
-YW5uIDxtYXhAZHVlbXBlbC5vcmc+iQI4BBMBAgAiBQJSBfCrAhsDBgsJCAcDAgYV
-CAIJCgsEFgIDAQIeAQIXgAAKCRAjbopYxttFEhCzD/4wCFZbaVa69wojbDGNfsge
-eNcXvCeLZYiIzkpgTCD7CHFTHRqnw8XLa3nNuO8xZbb2AL2Hfl9qlY4I75192+AW
-BE2mMqoYlJpiIMpNhT41NlK2nDInu+P7qkk4DgTf17++jQQfariNoj11HPtkzMg3
-km+srJhT5cXa6ug6E4ZjKO45o2A+tDY7YIu3Y+noTGB6UPhgNBLVEYNwt0FWCSEq
-6hpAnl1BimrKRMc8gt8rIjPBfPDo0Ys8XC6e2CqGhPTFb7j59G71To2QnRn0bS7e
-ZEU7u14MGpd4p0KUBP2rCRJZJc+VoukvC+burnh1MNK+lBtkZWRFnb9aTy3Ha4A6
-ffRGkALdUYNrxcXV41XLnZ8hRWXYCDBTWHsrusE0NFGQ7Cbq2r6Rlz0BtyDZxzAT
-knzRDKP1Ia8M9xqetKeS6hTg02tWOzolZw435O/oDCPWDUU7xKmPA/BHE6Q3z2lL
-fg+7aufceI+Dziiia1DJD8lTGrUc5gnwuPkldo7umqpmj5t2dhns+1iPGLbgnJI3
-pO+PcS0Rxm0A4PhyCqLVUZCLwyrLStWPOSp7ihA8UPleSfnw10r58UvipFIl8kne
-r4YPfzyNE3yZj+7vF6XS2qoIR2ui0H9MjXTk3FUocCZNogZshFKF1CsovSZy6hUa
-dY3NY8XQQ7lEHdyiKDYoQLkCDQRSBfCrARAAsSQeko7KM8fliBKGbNaenTmGEBQb
+sp39E/1Lj47+X20NGTk5Iom3xBydd8vftEHNIE1heCBLZWxsZXJtYW5uIDxtYXhA
+ZHVlbXBlbC5vcmc+wsF4BBMBAgAiBQJSBfCrAhsDBgsJCAcDAgYVCAIJCgsEFgID
+AQIeAQIXgAAKCRAjbopYxttFEhCzD/4wCFZbaVa69wojbDGNfsgeeNcXvCeLZYiI
+zkpgTCD7CHFTHRqnw8XLa3nNuO8xZbb2AL2Hfl9qlY4I75192+AWBE2mMqoYlJpi
+IMpNhT41NlK2nDInu+P7qkk4DgTf17++jQQfariNoj11HPtkzMg3km+srJhT5cXa
+6ug6E4ZjKO45o2A+tDY7YIu3Y+noTGB6UPhgNBLVEYNwt0FWCSEq6hpAnl1BimrK
+RMc8gt8rIjPBfPDo0Ys8XC6e2CqGhPTFb7j59G71To2QnRn0bS7eZEU7u14MGpd4
+p0KUBP2rCRJZJc+VoukvC+burnh1MNK+lBtkZWRFnb9aTy3Ha4A6ffRGkALdUYNr
+xcXV41XLnZ8hRWXYCDBTWHsrusE0NFGQ7Cbq2r6Rlz0BtyDZxzATknzRDKP1Ia8M
+9xqetKeS6hTg02tWOzolZw435O/oDCPWDUU7xKmPA/BHE6Q3z2lLfg+7aufceI+D
+ziiia1DJD8lTGrUc5gnwuPkldo7umqpmj5t2dhns+1iPGLbgnJI3pO+PcS0Rxm0A
+4PhyCqLVUZCLwyrLStWPOSp7ihA8UPleSfnw10r58UvipFIl8kner4YPfzyNE3yZ
+j+7vF6XS2qoIR2ui0H9MjXTk3FUocCZNogZshFKF1CsovSZy6hUadY3NY8XQQ7lE
+HdyiKDYoQM0gTWF4IEtlbGxlcm1hbm4gPG1heEBtdXNpY3BkLm9yZz7CwXgEEwEC
+ACIFAlIF8mACGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAAAoJECNuiljG20US
+5dEP/i2UpA4sZv+VtgGUL4U6Geit3jtcyj/YXoncLUsjba1mwasyWd6AVFk4nf2E
+Fpl4rWFLdLyGjkPDuyp0/Dv4F30YtEr+Jqz4nXGk9DQmIV7hKV5V48cgUg8NEjN4
+4hUgop7K31xUMiau7pVFXh9o4dn4INuP/LxRRyYpVnDDaNUrtb0nNVXJXRAzwVok
+pnaKHiHovu6Jgy/m5CwY2NHrBWNMhRMYTIzhHTSesDUtZh36x9DOyEffZMSbjbJT
+9QHfzk8vJnpGBxq7oM+o7oHk/XIf+ggteMj6d1CwbuPepUzpUQsqvw+BmnMEz7Q8
+j+GbwWyLyiMya+/NV+GO13UIVaeC9XAkqQYvzYYQmSgGhRtMKmTRlt1phES8OHRo
+wYgjNp571sY+6/QT+H3BhIpmT0Oex7aPxZ0EQ1nGGsGS2CbvX8nqVMln0+FgCYk5
+O8AKlPQqr+cZn+aCrSC3OGsoSthKoVPsrAhQcid1OM3agUVT1tk2VlkWJHE3sbEA
+lgUfMDPz/t8zlHapbFv+NkgvogVZLhGsMMNYO4llrDz2Q+FZwwDvwJDvEKl40xQo
+SsKWX0iHPaVR1Bl+SgC35Qa8C8J5CQzsGpyRX+chU9tBHaQxApTB05C7my8hpa2g
+HF/3qvJps8eGE+8VlhxDTHqRzE7rsOguFY7IN927iO82YAhkzR9NYXggS2VsbGVy
+bWFubiA8bWF4QHhjc29hci5vcmc+wsF4BBMBAgAiBQJSBfJTAhsDBgsJCAcDAgYV
+CAIJCgsEFgIDAQIeAQIXgAAKCRAjbopYxttFEprWD/9VcFzn+WRQGaWkZOnu6nSp
+5yKT02ISkpXJDkAdHWqL5U0F8vdiOUz+PpRjrjs/+3aqktOwnszGttfY+KDHQucW
+1qysuAnWScT4Zl2phbtcHVOI0GgDxgwZJ5kG6PNSkqyNylTZI6E0OCaT6vsYfaVW
+AZTRGuPqtT57/AmPuXTRD9Wv0y1QfkxTWiB8zpyqFyPXWm7rzS9WJ13cD5UNOJAA
+v/STiK2ndwS0mT9sAlfQqSwVme9d5a0Xfy4rr0/gkvSgYZbiVFl3AMhG4++bX/PS
+VTuOezjRO1tkDu/3/EfwX94k0eTNfvOPqpqirgWYnZvYBdUg1r56UhBZn2E90QRA
+YY4xY/XfB8hqLEgQ1EQOd/xxB7/AyBuj6UOKcUfiCZpR7bSozSFsWhC7e0VFXMMp
+vT4mptVNOwFa9AA2zwFI9VIFk/0zx8PAdV7uDQ2axfiW4zTwZRVP9azIygLPRThW
+UgdpXgSnxw6+u3b7/EeUQ3d+88LQ2AXQUksjgzJ8c5ND08oerEDPVbPIOyZh7WqS
+XlVjmtD1TYgghQeeFQArL0qcWHz+rxGCsxnYIkQzujIB+j2BxihJSVrPTDTNkrQ5
+1EJ1ylANt/P0XyyownqqU0ZufOdKqoqjhlSmRRa58Ftdkz33A3opyy4QX7VYqmfP
+t0GBIync2MJ6yliuUL3mms7BTQRSBfCrARAAsSQeko7KM8fliBKGbNaenTmGEBQb
 NcJtYYr8VJrkro680DLp3zv23oDn0Oe4dwuhZOAdD5Mgb4xTzRIgdIU5iSCj5son
 GKlCEF2qyVsxOWUvpFoB/y/c3kPuKkP5YobL8EHIdIz0FTjA/115hYNusN+Z7BOq
 xThVmIgvpV3GP4BRt/f9UWyWIcXQ7pGNTP9BE1H7DWnCMccG+lp+X9Pod3KHx7qc
@@ -72,7 +72,7 @@ jL09P1WBvFSgFoAetxa1cm/rrVMVm2vzJjR27t0Nu6Fx1WlQxelw7JilB2nl8Bpb
 RHYyj3WyccYxiSNbYVzYC+f63Q+a9tx6nSOLlA6ZMreOX2IbIj1+jq77kAWiQkOt
 +8EwsYj8s4qs0oVpiRxD0P8Tl0KOcZdE5EzSqTV8qkmug1qNcSZXn7tBDg1ZcnUZ
 pDOG7pxVCxYmB7euSGfibJIlJCDioWOnQynZtWkNrfaoH0oU565R0LfI6/WQvBrH
-/Gyis+cL03Y329cAEQEAAYkCHwQYAQIACQUCUgXwqwIbDAAKCRAjbopYxttFEkkH
+/Gyis+cL03Y329cAEQEAAcLBXwQYAQIACQUCUgXwqwIbDAAKCRAjbopYxttFEkkH
 EACR7ZcXaXhe7FjBfashp5UD3ZbI8ck1XlmnOqYH8bdFnuwTW2eI0aW3EiJUX8XA
 QPjfeYuQItj8GGMwFxWaQjatdMQX8E3X0V/FENu5pfUuHQAXZSbLaDgbBe4+eD6m
 JVENIgOhFNV2FX+grdjiYf+oXSCCP5NW6ASAtVX2NPHStRZweOI7kcGdcVZv63No
@@ -83,378 +83,573 @@ xDhP/q+5XuWK+Ka5qy0RoO4GlVWtmVbYYJl3GLOtLIVAVugIofGqbCYxG1v4HJli
 P0z79q8oJ6J5Iy9/X4bklnGvzA1mSKwUa1pPSxEsxJC0iAi6oGIfkEgebBmbmH4i
 jQKM9jMyJdE/0nZfAz+lvYvF04fu+68uizKEFULnCo8U3pXWhjV9Q6YSy/MhkK86
 myrb6+DIT34ug3mLh6rxynhWFEiTXbUVgx5rLkLLwqmqKj6YIStdskmq6TJwQ3En
-0MvWnQi4Xz4nT1Wdo0F+Mjv2m2Ah5d5C56Q4mZiUn3KxH5kBDQRaVNqRAQgA1jRT
+0MvWnQi4Xz4nT1Wdo0F+Mjv2m2Ah5d5C56Q4mZiUn3KxH8bATQRaVNqRAQgA1jRT
 gkCz7OFOXDTUlOo5cIVR+4IQa1w8xSWHhCZdp/QL0FPt92ZNJjnjhlI+J1gWj7xX
 dZEi8x23E65WZvr4TaY2lf5N0lOdEJOx4bxmiwBMq1koIOqnNo9o3XsJNO/ejqx3
 8G6hO0TwlZdQNbM624rmXmJ0WAboZvM7Y7FY+NfWfHKqvMnMtcAOXo84CRbX3D7Y
 qXW1fCuTApIu+fclB60kjZiiQkaCs89SA34/ySiaj4tMrCtlK+7KnxIdSBkptwST
 w3rpk7np3TGgER4s9uZZda6CxUc4R+R8KOkMOEkedYK6c4fUUz9S+IltrfxOhEwg
-jGEVWAioPYTZbAJfTQARAQABtCdJdmFuIEFsb25zbyBbS2FpYW5dIDxrYWlhbkBp
-cm9udGVjLmNvbT6JAVQEEwEIAD4CGwMFCwkIBwIGFQoJCAsCBBYCAwECHgECF4AW
+jGEVWAioPYTZbAJfTQARAQABzSdJdmFuIEFsb25zbyBbS2FpYW5dIDxrYWlhbkBp
+cm9udGVjLmNvbT7CwJQEEwEIAD4CGwMFCwkIBwIGFQoJCAsCBBYCAwECHgECF4AW
 IQRpEA/7kOh6Mg34ZDy+w5AJ6DIaYQUCaPEdfAUJGAJEawAKCRC+w5AJ6DIaYTKx
 B/9LxgNXV/ssIlJfU+wtLEjU4jB5kvb5Dw3DvYLS5vvbpB2MXFewd+URwKSCH1R9
 sdEgXJgD2UEs8iWxPXap5AYmduvUcDgpbyDYdSVmkm5otH402Tni9CPxPuexSw1G
 47G0L6r6Y82ZUwW5ZkgaFBUxA+yqZrzoHhySaarjL6Lw8TfZODXNJA9oT22/xNYV
 Fa5P+AkP1AGaLBWM7kl0afVv8PHjrsA/pK/wAbSrL0wHTnLMIZ2ITX+OlM05938Y
 4F/H5CSt+Sco99OjtziBFo3/ZQPKX9mCfcL2m8gYLMUFKqW52WVhRoQGJU7QV0st
-yCqVuBEmIZuE/uXUM7hFXazoiQFUBBMBCAA+AhsDBQsJCAcCBhUKCQgLAgQWAgMB
+yCqVuBEmIZuE/uXUM7hFXazowsCUBBMBCAA+AhsDBQsJCAcCBhUKCQgLAgQWAgMB
 Ah4BAheAFiEEaRAP+5DoejIN+GQ8vsOQCegyGmEFAl7GW7wFCQ3XgqsACgkQvsOQ
 CegyGmGqFwf/U7hVh0X+kB0xQgTHtTSuz7+0dMPWcjKsi7i1mHJWnni/juAQaLq4
 2KxZfllfi1+PAKQn4hdzzt6gZAdwJeCiPXmyoJtHkXfR9lZVslGK/N+JUKrebosG
 mSnryczaXF/oE1b3m+cbDtqztWxaA75OjtFZI9dxz/1dmcO91k3Ko3v5mLV3Niyl
 xFqu/PPW4kijeSXXZWxVSr7r/ksiusnTjUcBnqXmjaTsyfJeIq9qr0hRgkDpAwUA
 0Zm0Qb9YBicRcM0rR+QI4MvsbtJl5Geikzlev9DlCu7M3DKfTjaJ0bFkMj+SGaBo
-nipec0fZ+oKS/33kM8cbS0EiIsWF9OCW2okBVAQTAQgAPhYhBGkQD/uQ6HoyDfhk
+nipec0fZ+oKS/33kM8cbS0EiIsWF9OCW2sLAlAQTAQgAPhYhBGkQD/uQ6HoyDfhk
 PL7DkAnoMhphBQJaVNqRAhsDBQkDwmcABQsJCAcCBhUKCQgLAgQWAgMBAh4BAheA
 AAoJEL7DkAnoMhphxLkIAKZHFVAxSPRLQbs5YonXn2V3wMlU7y6/iTQiv6gjZjqB
 b1dtTnRYaokeGHo98wl5ZjtsDjvvmMGS95r33TGNB8w0yMOCeWMfLT11bIGJHwc5
 rbx7qkYk/lmlsC4Om/B9dqFJawHECTx8wKe8RDtHJU4jHS/0CTpSoXejOoF6HA08
 Ch02lhHkzDlwpVmSK8j14B1g9xw8ySfZb227DcWGmMXxVVdX+IKGzSOHG18eFCTW
 xfLSATvOLCe8fjsnb9Qv7olgQp++qAMoz09GBZT7pV6i2HVwb/eLtnB932nsWrp+
-Tx+1oTZr8VKy3eM1SZF7ad3nsIS+1mg5GxWv9cBJqlO5AQ0EWlTakQEIANmARtH8
+Tx+1oTZr8VKy3eM1SZF7ad3nsIS+1mg5GxWv9cBJqlPOwE0EWlTakQEIANmARtH8
 9yGF/3wyZkQm9SZmUqqDHWk5uT5UnOEOWtjNIxu9rHa4YN1thlFemgAY++7rCWDC
 T8ANANYpzwQSWe1qwfrJmMKxS+Nsfy6AiIQNdMk5pbGJX58smkP1LVwLUuZeG+nW
 CVEwdW5WSK0tOxDKBVB3EBhrD5CtspDjl1Mp0SKWeWBmiU5Bu5V2g+55qoWKR1Vo
 qa4223LzovKzI95lVvsyQsOWZcXqaReO3gVHW9WYRrtYi95hSOFFqCUZkh8ZvZIY
 CI04Qs2doQAy3btRKCCTTWkoLQWXzWsVIPdFAL1RybH8yU63FnRAprYvN1oe7FNP
-8xj5A9mfScgexlMAEQEAAYkBPAQYAQgAJgIbDBYhBGkQD/uQ6HoyDfhkPL7DkAno
+8xj5A9mfScgexlMAEQEAAcLAfAQYAQgAJgIbDBYhBGkQD/uQ6HoyDfhkPL7DkAno
 MhphBQJexlpNBQkN14E8AAoJEL7DkAnoMhph1CwH/2nhmf+NpcrIsSZQq/fTf/B/
 /Pp+4Tsr4wmkDEnUFpoUTbvNLWShcY7wQPKaKOK+a4raElDGjlCFPOpzmBW9EXwb
 cFtZ2z7VNwhDJk4gC/udV6rQC46T76XtusUC5cZvh1QuGJWMBoe09cX7D4oW3P6n
 g3zINQoaql+5UgPKCAIB6CiWRzkjRqmJI2DHUPcMJQZlEX9IAKR8L4DxYeo6o57O
 MMKZDhE+WIh4KMMqzJUc6Z3iTCsYbjO6by+Cik47HFjnO9P7vUioL+R1wmft1M2c
-nSXN0A+0dlYkvQ0qD/nV2eJ+rM0tQgW6IBWaeI1pei+20zugVas5qLeFsWR3T6SZ
-Ag0ET0EFgAEQAOMeMkZcM6Few+0u09WvJl41bVDYzg18Ty5Z8Ni1CLEBDx5Rzhh6
-OrAVw9L/KQYg+/LJPNPzDZVRDNwkYNpopSTluTVICJmnJxapLfuJtLumTHT/ZdkG
-ftq/f3ll+N6XMPq0hbefUYJRl2k2+Di4voXxkADtL2NicmpqRHHkD/SyWnqh7jJa
-N96jjX7MmRr8oVaHW40LbnzhMA3LGDTEJj3690/HWYXFH5ryK2q4XFrt4RyDAApz
-jqi3OlxK1nfDofe4O55mGn7+HKEoxL13QjosLLq1XBlqXfa64McamlNKyR7FQ/vD
-aEkyMg0WSw9NRjULZt+rUvK4IPl8oQF6GDqt/C8cPVePpD83jpbRWGbO9MekiR7R
-MwYBvhx8qr9GkzcM7fmykCz9TG/Nfv+VSek+3F5b8ZWFDZOCKQu+HHl5ICcYbCSd
-ZKt+d+vI03YcdjK6cbgko/m1vHzOhR2U930zVDMcSewblWruusItdPJQd0uWeEu+
-WPj0gDvNqmsgfBQ0a/AGOapsCNbjfrKlGNsqb94IQAlGA6GGNEjim7bua2HT3HYB
-ilxvnIpVOT2hWZJDcD2QXwBkxBXqPgUDwISxsDKRcvZ6NidzEihf4Lk+DSDNIKxx
-Hq1/+WPwrd9CyC4P5v69KD9RWkXQhxkSehJWgFRJ+PLOSR51YKDlSB9dABEBAAG0
-ME1pY2hhZWwgQsO8c2NoIChHaXQgdGFnIHNpZ25pbmcga2V5KSA8bUBidWVzLmNo
-PokCVQQTAQoAPwIbAwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AWIQR1f6t87RgU
-rhW0g25fsCdHQgNFTAUCaE2+OQUJGu3sNQAKCRBfsCdHQgNFTBk6EACe3DkZtG0y
-a0IzCT36g8da5N4eujTmnkmpvqvO3lk7Ky21LJfx0CpKx6EeY8WenKXGdtNhtMSl
-VwADMpYDmhVKUy7ooj/KNgkJEs1WfQ4JxmjS9s9hmLR5VyeO/TZxrtHbon5GfvaU
-ZUYCtg05nl6wxpdvtEFBz0vgx9NIqBLn4L9EmDCQmcRd7RmSfkbLWrv6eDzC4hDT
-790jH/8bl2IDbz3AreghS8NMSqFRmc4IKKVivdpdwp7/zRhKn9apKXNkpzdO2fv6
-chuLadXX7g0yv0UjMdKF/Wae9vtirLmAhWPKpCoDodgJmrYAFXByyge1nmuOTR3q
-M8Loqyt0hmA8lAPNJLsg4hsGlPSusbR3tpvcD6CtF8CqKNdQiAtKkkAZwKijzMC3
-CBVSp2+FC1Q71bXOssu8wBM+fw0D/XxS5iZZU7fD8q1V8pvwg+NhUFxE5yi2/IKn
-0vChTG3rMLMQcDaYtAozpbtwRHVURbGtiZ1kbQx/MN+FUyL9T6E77A+dJHRuSgVn
-Y2WQgxAnVS08GGITCuO/ukOpuxXhiFiZ2QhoFzaGCG+AJ99wRgKgnpMBM3VdVi+D
-DSCl0HJ4xLr7noTtVgiVY53smt0Ujog6W1GJ36dnP5ZCpwhV+DVKlYVPHtd3nk3o
-9MtyzpqSZz+2gcWrQfbpO2nS9rijTRroH4kCVQQTAQoAPwIbAwYLCQgHAwIGFQgC
-CQoLBBYCAwECHgECF4AWIQR1f6t87RgUrhW0g25fsCdHQgNFTAUCZh7EdQUJGL7y
-dQAKCRBfsCdHQgNFTDMtEACCs264ITN0JlbpLHlrQP8U0KNuXtFFoWpJQlabCqof
-AzmzWev0s6MU6hC1swzSLJ65fuvRmJcskdXuwofCS0bONPq0QtZ5M0rdixs3CxJN
-JFSMygtfJCqQG3YxP3LlNgKoRqXlLPbyB/SpnHy8i/0pBNFzFpj/5hteCpY4uxm+
-zqBe1dQ9P/7tLtUqAyEAGP/I7uH65t7L9AIoBpibJyCQadnBTK175TiB5FyXSErD
-rUkLNgRx3pPGS3nlSHrDJBc9MQ9NGhM9j9oL6PV+TPe6B7lje20+a3JkeVXJ5RhX
-EUm3dUPIp0KHGO6jUt0Lv6zfGYx/5Wnb1v4faE644Yp9G5Ed2cMlhdxH842KsZil
-T/cOzarPreUDPysEMnBAdRB5mJZ43K3kt7MDbBQN5X52nmJ+Dp7EES58ObCeW4mg
-JjPPGsxiOeugXfO04PZxsoTJPTjQ/NDC1Fnw+9ptKcMOIExXu9p5nezqZjxf1rgb
-ira5qXGJZwYdv+a9hzVSSO2IcZfrJ75TqoFka3mplhVf/f4nBgcApaoBD1qcnn4Z
-YvyjbCSy2+yVFeLqLAyUnl9Y1mtGWOfEKj2M4f+e5QuKBwOGxEWIMrW9HI6QjLfB
-38qax8/QPGHzQWng1+mYUMxBiDonVrOZ7Q6G+sqIBp3pHecPWA9xe9fuxNg9hHH8
-fokCVQQTAQoAPwIbAwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AWIQR1f6t87RgU
-rhW0g25fsCdHQgNFTAUCZDsQnwUJFts+mwAKCRBfsCdHQgNFTB9pD/4j9ohc4+st
-kkpFu1gE1UMJfAwPpFjxgmU3yEyR7+GvtfH0JAD9DMJWkuaXnfi0noqanWV593md
-DrGTqoLnO27Jte1WJ8u/SOnueh603Mj+/hgbN4Q6Sqcmor6kjcccL1zKiXdFc7Ps
-rM8VKNE2oWOWR+KDvSpoKDMNhkUelF6bSdVYsh8BGZPltnJbLkpDGN2ZVITsRtZb
-nIHVuinH9SUpZE4fNUTNQWK/Ob5HnITDxRaaez42BEVX1HISBONS54uYGpGnNByV
-Lf/hfU3aRZ6cceXYsA9R94N8d/Itg6Up5pfrT0mjJa0zELRWli2O8cY4DbFwU8Ig
-lA91Sxey/wTisiWwyGnxYsi/YPN8lGUYzgXbtDmflwWPiDIjGyEwcs/i0k0r1ffJ
-eOpctnsyDzCdr1DDW+F1hc13E1M9EHBUSwe1arvjd/0Gv+xN8Idrr2X8tb+LaHh6
-/XEVNUIgggVor17cy9h2BemYL1m1dt+2hwvjihP0ThQbmSwbvairpG1JNcrM6Sye
-0JX8f26WN6JkXGfPzjpQcKk99W0ynynFNKc5Ou72jpybiHfHcNU3mU4lY9BUjcmK
-GyB4PcXlZtN+T2jcTHo/WJS+JHdaGpwFq5W9Fw5WgSVKIrqPVoJP41M7JAjjRdiq
-QDDYseH8HqVEDRhsGs9Ca5e4peKBm/+eVIkCVQQTAQoAPwIbAwYLCQgHAwIGFQgC
-CQoLBBYCAwECHgECF4AWIQR1f6t87RgUrhW0g25fsCdHQgNFTAUCYGSgLwUJEwTO
-KwAKCRBfsCdHQgNFTL3bEACNOOWzlyxCHo2wd99mFKPc7+L4mJfkBv7eVZKEKa6I
-xIHg+IUqTC0jeYmEN30WkwM/CihQqZOn2LkbCIsUpkQv9jJBNGHBOGlV0w3rKR2Q
-Ti3F1fmJiL3rw14u8yc817BlU6qdrpYCbeMriYt/1QIrNuOt2fktoz4NTA7KxtqJ
-wT1CCJ6uIknpTcgh09W1AmNt6zF90T3PvvVaKOFXkNc42HMLmbd3HkUBVZeR7gQX
-Cs+vpSBKYKzvVmPb+exnTxMze7NThNIFHTJ/XFV4gU7cLXtlvzSP4Nr3+KV8hM4p
-VmyPle3PE/RZjUxVECT1OyNAfiUx3WVQWy+2RoBEfaDh1BQrEpd6zfmrJZQu8Tuc
-T8hI6LsydD+EK4FQoIbN7u60LXcWRFvflM0UYlhWQIkxQsjZxpw+fUa0kpVSRUEo
-/gyvPALmx37nv/z6xGfIrBuRMsuOzmWiUf2PcYZuWUgqqnHzU53hqQTePRkmZtFg
-+Pa7Y/LuPIgN9hAG4EeAvkVFbzotCl+S9A8N6ZEKazLukgDPIPYHEak/hW+GSBOe
-WeIY4zzR4OOZAfziQCOrY8r9QIAXs7TvaPZWvtvdprvDGvNkkLZMKVufPMQErqrX
-fsXe0C0j/6MibN6HHDYMaOk+uPowMhHaTBofL49OGVTFPbHmMdzcxHgaH9faouIZ
-tYkCVQQTAQoAPwIbAwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AWIQR1f6t87RgU
-rhW0g25fsCdHQgNFTAUCXlFsXQUJEPGaXQAKCRBfsCdHQgNFTP/nD/9rb5V5egst
-BTP1DkrQsntEAFsTY2GHP1iJMV+l/loWTnYe3iBouAIzPj7HQ1DIuzXun7rXbYso
-8C/NojH8TsoF8QLxcYzi0dOA/4CCEzfR86s7xsWTTVPUIWVzQYcY2OatH5il/+3l
-x1l0Eo2F5R7tl7Ym1/Xs+bvbwFUKsA1ZoVSVC2vgzmT50n46SvoqVJOV/43Cbv1m
-IYgOq9+P2tQy3FV0XWbJfdQBFH4HPenvpZDsp2m2EI8ZWtEsPpVf5ZrzTtW1i8E7
-Rbd9vg/v7ApyPbALR9fcX2ku5088ONGw92BYIe6a0fkBTLNO/W0MpzyEBczIrd1l
-vbxTKP4O9aAMFbBvs7NYNkUnN6Cg7yHusKjq0FajfbNWCM65RGray2mWyMok+76x
-WpZzMBCpPQqTwK0ysRqcOyhc3m3GFu5cEV5KiRwzs/Szb5S1+atT8CoQNxMoSzqb
-W9CaZ7SpOGEZYDde2jkYs0e1eHwdZ8sQy49zXC/hlyeZwMDtiVpcLJKAu2V+GRAq
-P1lSy9FSkib5ar95l/iesbmDhWRkNfsp8UB1RyKLg3WKdh/Y1vHpJY5S7yITtCnS
-7hmw+gy/kOC4/7dg5IHSsJjgja79lm1103N76PHIi+it4djuj12sisNHaFReASL9
-cILQgxAaUWSlWXWq11kng4Ttau+2FJr7FYkCVQQTAQIAPwIbAwYLCQgHAwIGFQgC
-CQoLBBYCAwECHgECF4AWIQR1f6t87RgUrhW0g25fsCdHQgNFTAUCXHqutgUJDxrc
-sgAKCRBfsCdHQgNFTIIJD/0fImQxyZOMLYAhCt8fr+5qRrBDof3ATwD0j2WBYDT7
-uVhHd2aZMfCyiZKvnnPyVoHmn8c20RMLXW7DnDppk6PQkpT5PndMFHScDAlx4Frz
-CnibGPv6Dcuzs/Q3K1hSCuYONctVxzaiMl2ZtCeXvn9g+6vXIZsqeJDNHNfOBOEk
-sbX/JoLxavAOAvDuRoRfn5DqyJu+CP6xuWTVIG/FSwFFUpc4JD6IL1BJ8FhzWXnE
-tXFBlhPVHeAn7FZhac4SdVP58zuN7+olAPQJ+UkK6lJgx0AJS4DaMUSh/eh1d+BL
-B0SlmGlxx0FPXMBLPdgOG3qcEbysyBf6X/CB22+ldw6Y9pjmpdnjA386T5hmjBrH
-8U9h2bDyhCnhdabyVP9spsgjDqoL23vSon35qdkC0ZkAhY1FEzckHYIHVP77tJYZ
-iFFmsP7tYdreMV+P4R7X7nKOSZZv0YXn2QY+aBrEGuMtMeHuqeu0coZBNhMsK9Sl
-ijiPuZ9BkZ80FnWA5X98yrEsVEpJNq9BN3UyyyIMmaWKxijLNKSmv3cVObL2FzOh
-3mvVRPBcNf0q/fE4cWGUbrjo+iF+Pjwsl5Dnr8rYUVV7zbm+6r8hxNqCzqCugIf8
-yzUD9fDzr/kc6RTkDphmxhO+ynzpESC2m1WmQA1VbEkebLlZaObaqUlxe38aD5CQ
-MokCVQQTAQIAPwIbAwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AWIQR1f6t87RgU
-rhW0g25fsCdHQgNFTAUCWprwDwUJDTseDwAKCRBfsCdHQgNFTOS8D/sE201q6mQM
-FB3WKpojn7e7IyUP9KPW9BJfM9jTq+ueC6ktEcZboMTrxH3y/lKBTPFUt+5lHi0/
-8TGcNiBaXeGThXpJcBH+Orw3N/QD0I2eRoZ+xv644auEAQnVPXY+iRnkR1UcfgT8
-iXuw5N/WcdPF+Ch40iNc+naSpFwqz9WNqIEq8bZHs4abrQpyArUSB0z7WxkemDBq
-Oo4Qx8KgthRnwbZMbqtl9om3q5vDfEh3y0hnQg971Tlpv1gI988QPyHLelv94ZmB
-iqqeeXDbwoQvTkGSnIq6S5upp86h9xPMdfay8I8/OvmrD/v+p+zLZI+mpPB7hPPe
-QD+nGgD6nufa/sBoGh0d6ag50PhFMoQ5Tyg1a6TLX6X7FSfVxvals2HOh51HZNWm
-9Huprg0YwWKFqT11Ggpzi9rJTE6HlE/hUv03+q32iT1rdpOIt+1FnEoikdxhTuvc
-yFNNKyTUGKT/c3rlWOcv2E3bkXWrY3i2LYzrXZ2I16249fMqoy2Yv/+tmWfh1Pg8
-TjXsbmRXgvFZfYFtO41Zuxn7ZaYlaLYQOapASZOAdE9u8pqdkpP/TxaSBk4v74vb
-3wpROYma/i38SOFG0DfxG79Dcvy0WwDAbkWZ5lRQzlmUU4jvShzcIUADNqNRKMLt
-EGJabHNRhyD7N3UrSx7TPoldeCpZr8v2/IkCPgQTAQIAKAIbAwYLCQgHAwIGFQgC
-CQoLBBYCAwECHgECF4AFAlbYe7cFCQtZ3TQACgkQX7AnR0IDRUywaBAAxMSdqm5h
-QC2XrS6l12K5tl+8wIlD3r0enitZFGewy5iUoLeawDZZN08fPrqw8cqWSEb5poV6
-ThBGeKpCIYac/m21kBnnmahq988ugi2SkImip9WL5bwg+5xlE+zrgJpMGRSdtQI0
-Iu1Afr1ah4rtyplmPZJ6T/AvBGLaJvwRYouhlqQvpKfRGwkShHQGHKgP+FfXUaoN
-S2T6feW4atbjf3wrUExWfSr/HgEXgTD1lr6JFxJmG0Zntzcbr5iDSoL8N9TmRTQp
-3PBbORcOt4tdBlwT/N++hu/IeydGER8clx1rtgoBLD/kWyqr0EO5iqlUQ7etWHKn
-7UF+Rim+C6xZ02gWdSbHd5klGa3MkJKjYLrjDDUZkIgUjjmYjCB5+ZkeaKDzGR3w
-B0kGCi+ofN6d81PPTVLCVevJJlYS5ol4oQJmnapaovWc6sXUPzbaR6Qu/VUBLygV
-1vwF+SP6lAW2I7HMQJq3y+G4kwGsYroKlw41pjXJcb3dQ2WsAv8O5xwpoghdzm3F
-b28550e9FTHo5QcLEC3Iv751hgneS+Jsr0Ehw/EW1GrfsZWAK2URCp/qEEupvJ5k
-ZoNM8LdozZ0FVNpLor++OB+9bL0uRXrWVYXNkzF9I5d9dXUkIIeSeGlzI9SixBPC
-c5EX4r2HzwXNdhu/JrykeLVjmwyhRBEIumGJAj4EEwECACgCGwMGCwkIBwMCBhUI
-AgkKCwQWAgMBAh4BAheABQJU9zRuBQkHl2JmAAoJEF+wJ0dCA0VMnfYQANQQiyG0
-GUIo1r+bc/M1OjF4fRokQm5KCxLxsH1WXV1bEaLAVUxh1vwxTd56dQ3pc+tUcADr
-AbZlXjQbjJTP5GCOqJBK7XVR8gs/OJOgvkcJNL95nV9ZkcWVu3vQCpI6RftabqXn
-6MTmWAjqvLMyIODAu+HL0VRdPQL9BglH70/ruCghJEfZmWBfOzfXhRn/8ad91uDu
-RzUBjT05jp4tHpIZMe+Jok/pDNkquRytjIwszyT5oe4YG2jPVnJ5yEND/hAbDQNk
-ASFaMzgoq4YhLInlB8Rmoqv0Vnkpk6XfSXvZSIhv1trVHJg8AXldbehGixJBTgmu
-9R/j06+Sn3V7mHsZAAYTXJDXnByREOMRYg0Zqz86Nn027Dk0htp0C5Iy5qml4iMM
-crjN+kmO2K/rKzl0bnAiXWlERNJkMtZaKSswTuz81YCdPxNgzc8/P+g7B0ryCf0g
-NpCVPOtX4tajWwQhRi5v27G6uluU9cLB1tttXd/TfJh4xLPBt7XD3NeKnFCvGTS/
-BzCZKV0Sx7yYmn4r/y1R7Dec32NJIfmdUTGzvWWqOJUf+NkWD4oZ3TJXTmpBzMAS
-WUB/wu2+XZtJCcHEXmZt7upyDzsRzVg3vJ4oiBHIrdwpPiofZnecFoF4GduSjk+q
-kVgTaRy+UDRVNZZ5mQAXHGHqpknYcmvZ9Z7YiQI+BBMBAgAoAhsDBgsJCAcDAgYV
-CAIJCgsEFgIDAQIeAQIXgAUCUxNyKgUJBbOgJQAKCRBfsCdHQgNFTF9qEADDWH6u
-5e2SukfNCTJogeSJj5p3pELQqL76mx+1yV8igh/uMfqejOIUbyEWe/KQR5pO/lLs
-792kG0Z5gfYv5SIEl2Y4raRJ6k72xNekvC1aPVnspX9YkOMiRuKTYR4wudpqONHf
-g3dphBk4jzy1hCKrBG0/eLbeWazKhheuI40wSaY502jPANQ/vwka996JZxKlfi31
-hSB/01+zj7CimKQmnagApbuv+j/m9v8vNtJK0sgddaPkJ4JIznmulDCUUm060DYf
-E4QpnTuB8/kDJ+c71yBMiNFgGGHXVawzn5G6uFsKHwV5GBboXHKD1URqxFTDLMXt
-GM8TCNp10pDAFTxJB+Y+/rUjv5dQa5KQY6A6IJ0SabPjZ0kjgQbUP7n4XXu4UFzE
-apaVpuwZBwSs8y4GaOGONK22tE0vhuuPWmSL1imgUJzEucT93Q3gcEsKLM/6HEt5
-9dDsFnYsORPkLX2Sk9z3ZNQljXZdGoU/oOeO+q4ngDnKYX73widYvJikjYqDVlCW
-sRCMsth+omgav8ncHKTwvZ8bdt6YnHv64bijhk9gW2Ifb8kkmPZr0Kp0m5rutj1E
-Yl693l0LimAOXAKZiGnzhkolNMJGt/VY+8Rgrt8mNvm04ZKf+f8CCMmCejIVhC4X
-0ZXdHwNttNvq9U2NVLdD6zW5claz4Fz+XjqoX4kCPgQTAQIAKAIbAwYLCQgHAwIG
-FQgCCQoLBBYCAwECHgECF4AFAlExFXkFCQPRQ3IACgkQX7AnR0IDRUzy5Q//Uy+D
-1OD2SWh34K18z2RoeAhZcf2p8IHzWX/Fopf+DTRMrcYTJmUWQ2lz9IX0iUyRYg+R
-uJwGsDNauYRukGoaj23kXntCKUcC88bCbgMR56oxZUlm+t7AFni/Nk0U6jhhw8wj
-bIVa1OzQgmJuJhRi6VRiOVjBrzIJzSFeSYyJCrWCcpAUJr7LT94qKCLGuoHdIZm2
-l7EuvO2GcgnTuj01MsLeNxxmG/5b5RPvYCGdYaa+zawidOdRxaYzPYE+gmo4A9Uc
-wFk1upsy5puc3m4REh7gT28x4elwQo2NzdvDqH8JE5eofxesYEnrDPRqP38+Ksoe
-00gYdw4yKnr3iu8mluHSHnneRcZVKIiycKpE01ONwZMH07DLa6u7VxUyiXxqZqII
-MMqcMtXNFOSyTA2V7r4MbR90ftpe7mqj/jzcnrzr3L93nwMX+pIpRy2gij/m4MoZ
-bU/yhBv3FDk7RAI4B+TK5EAch8Xp0YQjGXGUIi8JbAJ0G+at7XM7+rAKN6ydpId1
-78QJR5hOaWCuto08+CluKyZZ1xQZZwxwHRyWFc2Etuk8yD+RsEALdph8UTgmrkxE
-II+LnKKZPtWDzHMNqYNm2OSKB5idx9TOJyTN8ETmjCbopTJZDBOY3tmFYpERmXhS
-Rom9JvnJeYira38MRHDLwKMUjLY904UP7sh97omJAj4EEwECACgFAk9BBYACGwMF
-CQHhM4AGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAAAoJEF+wJ0dCA0VM60oQAMr+
-7xd2seBN00uIzOUPx2pLvU5TUj3Nn9JFpvaJLM3AYVGVCnHtQwItADFanbRLdOz8
-PIY0hnWRS8CQ1QAWeHT3wW68Y+qTtT4VKmJiwo9QlhTMxYLtKmebhwWf3Wz50xNJ
-LhYxVoKb5F+Hr9K9Q15AdCH5AWQVo9m4kIzddjoJL8Iot8CRXg41VYoh20p5A9au
-UqndpQnuf7jbNyNsGpsvo9eQbyw7zkTK8npMQ0374AFqdgqTlBXl7OxxVZxCqVBF
-YnRb9d4l9wrZyqOvinqgNj2SglKeL2w/L/lfSzNvlpWu2yRaflK7shkQbd8tKphc
-vkHmZd6LPhN7GKV5wTQTtzHoH7Jp21M08m+NsihcEW2RcCQ4lj4IX0SXDAnVONtO
-Huzr0hqU+JV1/H5bjjDpdXsxD+6M3nts45hc731C2WcbrjoSeCFg/UPWpsMES1Gp
-25xNIP0tbk5Zo8t6EKdJYJCQXqi+gnPQpfuyJ5rY5TRN9+9LlLnzRB+iZNrbvZKa
-+mPnlN5SCESsBKIQGL7GMcMVLeTaMB13Dw7H1utncdBLQ0e84f6ZzvfFzk9EEOZ5
-WUXtpCod3zqFb6xoCfSBsLnfb3q86NoKHX9AeTuIdyFHqkYVyDAuWKijfJhQlbOm
-X4UPEADZ/MEwQ9KSb3BbLQ0ImBuj/EG8jwzWW95ftDBNaWNoYWVsIELDvHNjaCAo
-UmVsZWFzZSBzaWduaW5nIGtleSkgPG1AYnVlcy5jaD6JAlUEEwEKAD8CGwMGCwkI
-BwMCBhUIAgkKCwQWAgMBAh4BAheAFiEEdX+rfO0YFK4VtINuX7AnR0IDRUwFAmhN
-vjUFCRrt7DUACgkQX7AnR0IDRUyOLQ//fVOhZ58wacXCfxh1QKk7hemNCW0CK9yC
-cHyg26w4u8DWA+X6J3i5GaGW00om0CTmkUrOju3JRFO4qfz6WlB/EHM4NiObezpu
-MbaHz8MGXdIibf24nDl8BukeiHbC9cjBLYYjhIAA2glrC4pRFi6TWpVn0/hCR19D
-kSFSt59HBV878wk80GJeHbGLS3Crnn/EEFt5ytGFUcrgH8DilRJgCCK//8HfHAl4
-gjq+oQYrsx54wrqCObb16hfnap7TpC6l2yNchMaORiluRdwHKScdflzddGDXoMck
-7R0H0+QWvqKEDxPYjxHV8k6pNau5EtF6tkgjViDA2LUwtZR/GJTvq7N3pDRfB0Yd
-HoWNO56XlsD6FzuQAjfkaWMvVsS6p64cFP+ODFnTQdyemhcR3mWsR+NHanC2C3U4
-REi9kCC3QRjvNJVKKh3ijmvhtPIlxgGrXG0p4FwoMuXVnWAThK/NA6pDeItYx8A1
-4B6SwhMEdud5E8ftgdc+k1/5e+ybcv56OfUB9N3Ie91zh+QZxthdqGogIOYi26XN
-kvGXLAwxPFtlFI/XVayqydmBIRqPjQ+AXamt9+W+rcBS3eEiqlIheI1SLKV42Mcd
-X/nJfFn31R8MwU9J4u/w34JnfUBe2dKJIegda9705uXGO9vNZfGNgbSUH0otgy2f
-bYg95391jAqJAlUEEwEKAD8CGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAFiEE
-dX+rfO0YFK4VtINuX7AnR0IDRUwFAmYexHUFCRi+8nUACgkQX7AnR0IDRUziwBAA
-vXeAgmemwA8pke+AYQ8cHPR5/0VD/x3Zafa41dcE9HsViOsICG/MzUdGHoXjdKFh
-5hG7gk/Mk4oSib5zgk1WRX8hxOWLiltUDNrI39iYVRanAj/wrARVlgXA+Ouw0h7O
-Tr+lEzjaQ+Z2b2ywzrmWky7hVmTnLQI0k/7jJT6Pjua9N+kOt6X/eRHJxe5MSYv6
-kQjdSQVLVz/KUOXsieW7lutRF/uP0ca1I9GPJSf8Jv8ShSDWPVnwVdcZCUfWceA3
-4HvDWkzk96FMsB1Gf0Q5uDhNvy+YNZL0BwaAs+mqalPm4P6iIKYeawDcSFa+rYRl
-x6Ydw7oeCfpjjeWoGvAfmHY+/SLf9XVv3kk6G90vacbWnKrJk3me0cl7CgsLhKsY
-QOnpsP+TnuUd3gHvK92bAniJyGKROp5CofLWJJUZ+DdGQ6wEG0Aa/ZdO9vMYZ+RB
-8Yup/8/E1x7Fdit68kLnk6Tjn9sY8kth4ayKY2la1DLpUPvapciaJfStujrn1poB
-OAUmuDZ4ZFMrEbqy/NNx0FZDa2JsS9OVlAtbOKgRJLbVJLgq71rTlDyVEoqaMcjT
-7JAlsrjkje5ZUGqNJGv2euKdkkl/UcS+pGSyC2w1wwygM4OvYhgdoD9wFSTpHGIb
-Iy9ZNk10Befuzvy98ujOCzKNDljRippyvB/JK/+cYBGJAlUEEwEKAD8CGwMGCwkI
-BwMCBhUIAgkKCwQWAgMBAh4BAheAFiEEdX+rfO0YFK4VtINuX7AnR0IDRUwFAmQ7
-EJsFCRbbPpsACgkQX7AnR0IDRUxqNA/7BKoc4lJfusZGlYYqp43SctwYY46sBTHv
-DqEur59K0W8kI+T16ItD4MCDz8/bqLm7MTiKlujlTrg0h8TpiEz3GO5muDfzsQm4
-ZWFOgcuOoFBv2GZR7Xb2xnWTXoD9mHpic2DwWYU7clr4eRB8LB1rh2p28Uu0NQ4c
-2pbK8lD4zjicbU1dCgLV//0MU7rdc/b7ZWF8G5N3MHJ9olX8zR7imgYdmZ4wApqa
-9CW9aztLtas6gn23wA4fyJYtWNHqhX9zrLSjVYzoGuS+uRYV2UjhgFh+keDZCq6X
-uDYjpTzi8vmPSqu5orJlKYm/kEBoc3ZG6crpV7riYwELcT2nvJJobHcr0AfhJORz
-BMlolKFKD58uqPvc2qSpFhfsZTFGTUBbPMzTy8WzMOu8f+bySF6tskUJv569U9p/
-vXPIg6HlcKIo20FZNYam+BAWoouTY31QmQY5o3nnBhylcmTM+g/6RKIG0g8ALnYm
-ZvbqN/tIC9VQPWwEko5yWK02Yf3TQ8QqxdGudVC/+59RPlh22GPqYoMgzj7bNmHr
-hshKeIC9VESWbbZiFT2OvGfvwPivZAMZ9moUmvrHdYLzm4bSpezPMESQad4Ge6rN
-YJ8wiprRKWNrfV7bxyXZHrVtwOv7gmdj9Vo0+v0HePmEfTYf59sj62etOEsKoZ7Q
-JATM/3fJIeyJAlUEEwEKAD8CGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAFiEE
-dX+rfO0YFK4VtINuX7AnR0IDRUwFAmBkoCsFCRMEzisACgkQX7AnR0IDRUwxrw/7
-BZCD8t09fXtztL1jfHH85uRwMnL7TSOoN8qyyb6/nyjyg5BY7msEBB98fcAjtMpS
-IXw6UK4HlqMrfVN8lt8a7dG6kB5fKZGjKzDASQX6xyVjGkPdIUUmYHMcy/Dg4h1+
-FmpPwzomdVfejAfMoK6sL78eotGeyeKzrfTa8Jvc22nKQ5gstrpy5uOvhYcaZxlj
-6oHhTe4BvK/GcwAtrfqFJswnlalKrjT3mkwrsdRmv1DF4vTLvly+o0ZJZhvGoUAL
-V+ccGyisf3f2Dr6KuOzpYnFy5Sdi3N/mcg1+5xd58swVeCNbmScdrN/Yy7aNNjqO
-Crihh7jjJ4Npt2KNLWOMNi1N/opLWNwZKtjPILdvygyBFN6FGKtp2JuoAQtgulN4
-gxM7ABQnoBgwBUKJSvOSHlcaVIZkkkjPbre4rjlts03cNszMHivQCOWDRsfuEAQH
-4umPUn0tjvg99RJcW3VpNIA8UTxoG2OuA1Sen1qIeIX/MixZWgWo88KiVjPY4Dfy
-IVQOJi4+yzeuH0D1ARj3720Sl+KHLLHfzxb66pL5Rklpc/trEWVFRQyzVj5lYidu
-cLwzA5KweCu8YBXFPKuodXBmmBrAALFhxQ2i979AN9ypiXZV8jG/rULGAHyv7sFl
-DVmB5TDjzMV+MP9Kgg/VJc0BHrUyJXFhRJD+wpM1cKeJAlUEEwEKAD8CGwMGCwkI
-BwMCBhUIAgkKCwQWAgMBAh4BAheAFiEEdX+rfO0YFK4VtINuX7AnR0IDRUwFAl5R
-bF0FCRDxml0ACgkQX7AnR0IDRUybzQ//a0w8rKJYFrSkX9a3pemZbyf5+FLqzNBk
-mEv1NvmZOC+1g5hO4/nVySSZau+fR3pIjAa+XrRJYsTLbj8FBrpi63LYW8dWAXvB
-NoqBanxMXUS9u6jJnqaXK57Uv/CcUus1KiTX5ufEF0A/CEsP0U1m0ka4aUjuOoKY
-nZReX+ZHRXI3dqpxzStAo8zBJlNOYE9wAH7zDdDIs12O3EncsTL82GMGI8q4Q3dL
-zNFCaTLw99j8CLG5ALt/Knk9dA+PiU19MUxaytlsk0gGPhmaHUmZeJS/Qs1iFSV1
-CrfbhP5SGvtQGfo9ei+RrTD3izRfg1tJyDFHcdZCwtSMu0VN95YSDTHe8PutWgbw
-GVSP2KZwMA3JNWq7Q1JJ1YmqiBPsGN9Uz/JD/DIj5IdHgUt3AblrNFE0eGTf1UXM
-r+Ll5zw7oI+xL0VrgE1SExorrc2jclN/Bhph76hscCAJq6XXQK0MN8BXmTUW2Xk/
-Kcw5YyAc44GplFSmOKK7nipcUJlxaUV1mqGw6C87M2rMx8jAvAe9fgwBxMnIjjmE
-61+JxBYReOsyF0ogZt5sS7fzSNvTVWz6mnx+ii1kRQDlEnpdORbX6ZRwUI2uUsud
-ggZYxEh5Z/p5PPVbvDQ7FGSOt6xK2CI71fAcNnNmkRMEodA8Iccguh3ZOVVs8ji4
-OgkPOQPCGAOJAlUEEwECAD8CGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAFiEE
-dX+rfO0YFK4VtINuX7AnR0IDRUwFAlx6rrIFCQ8a3LIACgkQX7AnR0IDRUyTKRAA
-wlJyEwr6SdHt24dPI7DEXs0GEFijxsisl1aTwUuEyw5oetm7kR0WKCeZQkquZCvK
-7XxIKSHu/tsWR2HI7+NmjvIPQDmNIRkuyyCUKIju2BcMqaqV8bO00eqXX8MRsY+t
-UHijyi/66aSV8G/G2Ey+8uvF0cimBggRFRmBcLMD+Q7Dvy1wvDbyyAbptdbgh6r2
-kP1gvxp88CJ1/3FPNA6Ufl1iTtmcJ3n3Cr3Lq3NtHeDsdov0cKPCus2rOHwTW9wN
-EJY0EPk6X2+i/KjRAPHscYR/qg9AkiEJpFa7jxRJJWcX7bf3m/Z73b/Qf+t2/I6s
-u8zWJPc4GnDVefD1WpIbAuhcIWWUXgQC5gP6ltfk7/1RRykHfkypa6BAxM75d9iz
-tymFXcw6Xizlqn4ss0chd0gdR7FZ/cbsa7hx41hcNZdgspZ4p1QUntFdZvDLWZHH
-+qXUTXg0dUJ3hpU+5t7ajXUIxCvo2CmEph9WhjTgmfYl9VbUiXGm/vLkW2LlkfhA
-tM/0H6W3slgR/6x/yzG4Qp6/AtrqiPormyEHkkOZ0cy+K4JjwFpMxWZxyTq68ohg
-V6kCEAzOE1LoZdRkm0Ve6RER046Nkiu7SB04CIze/8Dbv4UlGVdR7QgHPUx1lim/
-dYx1fHHiCt9myjzyT8roOgOdGwA6UN/MnQ5MootwVVqJAlUEEwECAD8CGwMGCwkI
-BwMCBhUIAgkKCwQWAgMBAh4BAheAFiEEdX+rfO0YFK4VtINuX7AnR0IDRUwFAlqa
-8A8FCQ07Hg8ACgkQX7AnR0IDRUzIOQ/+OlIyuZpeCAD2ODS8PEtj6KDnsBnjWDDR
-mbfi95C9FHHeEEhkj3kAf+bQJdL5C/DASooHsSS9rNe9uSMSwcy47BGla8BoZ2V4
-ApS7EiDsAfDQ5ZhB0JJeV0dYxEdD8OjBH+xBjgj8yBL4JY4zMTxs2qrPLuoYJ1+K
-ex0EEEJmAtR9xPOJ6Jhb9VNtYZ9BpY6X+srEoMH1TOQH67H+Qtx3xZhPuFJexoKo
-OE+pMfwlb1DL/ZqoimgbWzKscawobxrARlBeWh3yAbT/iVfsPeqzppPuSnQkpBs2
-OSUfEwjYC4TlYg3idN8tY12tOVuGFL/7xvLEg+tbNXx0cv9GCqxr39Do3ixVyeDC
-/dHcN/YiHhr/CHwyvt5ZvrAjZwDDabIzvYgcO4/X4wRXX6QjYMx+1AtXPk/Nim4Q
-oUY0tE97Z8yccUj8SS/HPm1k7Bmn86lbpDe3cgRlw3rjYxeLU/0L+KB9DJUNl0ix
-T9cthBurD9FyaSCJeiit2S4u0A1gGvyRc9Gvxtny4+G1XNmHZJinywc8BODQhitd
-Fi3yohsZ6kcIOjHldHmna8DVim5ijhvcVYWGXQR0S9shmqDcdzdPIYytUw3UEH5S
-EnewSgFNmxd3nJOdFSDiefZf7i8hsZyCY2fFc9NwVB6cm4tRnt9Ebef+VAkmyfJl
-Ip/NqeQF0SiJAj4EEwECACgCGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheABQJW
-2Hu6BQkLWd00AAoJEF+wJ0dCA0VMSUYQAJQwCqj9Yxzhh1BwdAh4/WvCtA3V9hJc
-4uHnuu6gdBf738lsNwLjKsFzvJ09XcfdLQaDUFlHefUWNQBRIFlDFXFZKwK7kZpq
-lfz2mMV4+YhnbMqGnA8TxqlzQGlNBEfUxy0gmeqCWZ4kzVWKaK2QiNTtAUxA4F1F
-oQsAE9OYC4QIyS9RRePzA9v8oKDk+7D4yoRJiaR73i2HeG+83fk2LLl3Zu27Q1d9
-Jrfd+iOuRSweiAIB8Msq0m/UtX9VqIKCvf9cJEat7bTTNieR1VfhB0mgRhdrGUB1
-VcV2aJzGNWHMk7uESTw1jrSinyQyiaRt3Ay/hePAT85yAxO8YUD3DLjaA/pH/SpS
-f339/OP+EcajwkaQIqmANwV8hBSQxy8+sFp/Hanop+7VcVcg4UB3bRbJ14QdL4ER
-/A6Z3vIdUyEVu6hdHYoh8N5yG210CYFQCc2vajBbizcKHuR02jr3toNeNOu/5UOA
-aT4llqKJS0z4S8UyAmgoLLqkaBkRNDTYQJ8ZpVbNAmQX25wa1vxPE7RcL/bkiLpK
-6VKBcZnzWNhahq9rV/mVOdT48H3Us+OEwG8L8agTr2R4O/I/uOfb+/+dN0vF56Et
-dkt0YqJAMVXGnCzQaCFWvNXakBBLEbdgCqwYj9mdlCIggpIVEi7onN0PiFhZXQam
-kP4Ca6lXaaYPiQI+BBMBAgAoAhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAUC
-VPc0awUJB5diZgAKCRBfsCdHQgNFTJv1D/4+CrqUB8CEYwox0gqMCvY1/2N4ylpA
-C2RDFyX082Pvj97KvNTu1NAYeeyOInJSCEviEt9sUXCHGTpBV7z1wfVfJ2LqJ1bL
-iPSQGTksYHKEjyn/dDoxY3smpYimGfxNg9us0m3OARyn1jSULOpusX1Acx1S24ky
-3FM3iQvZ8+TskSPZ7VKLoUkmVrzaepy70GsogZ8xUlQ7vRoWU0xzx2GJCjB4e8/j
-YpqpxhFiMznKcSqUAcXuT7LeT8jG21361kZCJxA8ajKGcQ2JgV9W4EwFrBnvHthK
-cdbUQzbmT8wVcD4w6tw2P6KmrLX30/Lky1DC5qiuLxo/0qSGPUgKrP/qqWYRJSAj
-+e2Yqt3L1WYIThq3REClPN/gS3+OHFiywu2h+TC0SppQn15TphJSWQgRWYjumP1L
-ss7QD2GLVLQz8e3KnW00lN4E5TxR2h1FKoa1MxRGhYp6zqICIP3ok6g6c2jphw0n
-CJLzUH+LTngCtvJNybJ9XaCnykZ2ZmwGDHdMpVSxDnv2Lli2Zk1CGazFCmZkoArU
-thQuS5rADDgZdZSVk3KI7mzTEpRHxhEuY2k3q7unFzThphVh6HbmK/3XIkO3/yL3
-ljB/5DOFmF3xHwbaZa8R1TYlmL5Cdqt8TtajcX1/L/IsXJBjJt4Nkw3kLUzA9oWN
-khKmXWIc2QOPRIkCPgQTAQIAKAIbAwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AF
-AlMTciwFCQWzoCUACgkQX7AnR0IDRUzPpQ/5AbqKmR05yPx+un/hMThzgToHF2sF
-AH/2jAsVdW6KGYiRUgc/7KMFLLocFdJpCGMcaavK2EySozCZkhhkJeP96EM6wxvJ
-xlpQTGIyvr+Shlweajh1Ez1TbbJ0e36r99gV3ex0j0R4gkg0PFqXp3/eRGvBkya9
-cVSJbgGLvkHBS03/CFtUjidFCbLMvKCWrsr4DTOFT2xZgyAAG7zWw+BwIwCJuJqL
-eH4AN00COUgxVJKFHsJvN+7XWA7hrvqPXgtgrkGopCHF84NbkbikknYRiF4YR2Sa
-qkN55tV7pUgkursG0mtG9rAKZC99XvTWyKzH3bjOiU/UFl41cg3R8jxt6t2e+Wk+
-i0ZnP2ar/XAMKHy3R143UF4MZasFkQ89BbupsrQlV9XbEXiIIknjNzeHriqx2EFg
-yfwnWBhRsGvAevq883WLmKIwX02LrZgVRTs/uPoa9Fq8jsBzV3I6EtZrQcGnZxeB
-FlISREVAEI1Dz0AG6QBggk+3HEeetKfJoxXHMBRBK68Po722jj68pQU4KE0Z4X7d
-WKv9xy9suG+DInNJg+VvE8AFwmv3LD3xwoLXQobR37e2rB6ifJZalu0n2xxxRAx5
-VPhb708lUDvUB8tyj/qU421Y8CKy96Ui4Cwa/6BS/Dj6CIB2vyiif6ugYtsFF+sn
-FbGTEixX3gCm+dGJAj4EEwECACgCGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheA
-BQJRMRV1BQkD0UNyAAoJEF+wJ0dCA0VMg+8P/3m9xnEBWWm1Al+hIyCR4lkI9dlw
-Cgb2w/A0hXiVou28oxGXJjMVybEFHgn4T5oAvFioWxtHs27vgUTV7EVgUwu1pCiM
-rpOvjblJP/pguwycHxNX3V+mccKBOMOtKSF2IRMZuTwXLZXjwlLp7PJWw22ZyBr8
-AWwBEJSpDn+kZjmQNaeNts0zaEG8lFdlj+5kmiar8EyWhQadNfNR6A5LEu4lfmdl
-ceTJNeSFSQjUA073ZYao1/oZNnRElL3GiBWxHRmMFkSs9YqNtQ9Lvu295yizdLlV
-iHOksY3DtPVyuud1GAOfVMVAa2en78B0CDmIIsfA43QMyJnX4h2JFARk3pHFqwXT
-ubhlCwc7Qc1aD1Z4RAPhG7gLz2Vnh2YfbpHtjjXv56dywzsfQ/27xaZGCV4PGS3w
-1NVvHxOmAEYVDqCwle4MG5IdOeWhIi7pxTZ64VugdfPC6RncvXqTvSVerpbTPXzy
-qm0PXEZ7S/ddHcCER+z88AUzNN6+GOiysHrFNo5VZ5fEL1Yy82mvWvjOP3qkWVoE
-Fgqr3732zDKfxHO34i1xKn4iI1xUlz6Bj7KY/chmt+dfR4dcaC6VAYAOSv4MnE73
-8PgwFk8x+0y7JTh1C7ElT4CabELFKoZh5KTqda77ZVpp4N+8mUziJtvZF0otyEUS
-F408WxCrFiou6J2liQI+BBMBAgAoBQJPQQ/+AhsDBQkB4TOABgsJCAcDAgYVCAIJ
-CgsEFgIDAQIeAQIXgAAKCRBfsCdHQgNFTBM1D/0aDDv7n8vfFiCkynRHdfVcGtqd
-4ezCYx3dkzNWlFfdRlFs3JzTYu/57BZlcCycqY8P1Arjl/2P1HISt/K/PMhcXE8e
-xuymZdPCTPlAYuENmwAyhLtuogvWeV1M47YJ3R7iMa/bx0Y5MeaKT7FwJFQOXf+B
-4H5lToqrB48dTuqUhI2f/AyCF6hNejK9spGk0VOIgZUDuOHANFsDTUXREMJtS8Ws
-gYWoJUA77oD9I7pW2axmFULpITW4izmhb4lTG7/GCBsnq58DsGaTu/1MYzmz8ie/
-+//GWHxyJELQgighslJFuGdV3Pv2gM/57Kk/SoBw9T5eFhkYkKM+gep24foCj0Nq
-BfHMomNjT3s4cIkCB/772ybbqnja9JUQyFDTKP7vWr0yAmyRnzQOPKh//RtbhZMd
-19N0YUIH2PP+elybpuCL7fSZG6O8uhKFY0R4stQe6RhPu1FGXM7i6Tki86LeWW0b
-BzFCmxbbLPC7Vfu/GXIAuQlCxdF2sjF8KnnYMZDIu3WxzovFfIYH+Hmm8LYHElly
-D4wW96hC0H8wRo6clQnMXB2Wod4RplX5RNFYwVjI0/KsDndU7VecsVC1QBJHpi6d
-QdXifc0NgU4eLbJdy66x1rU7K8zRQhvXs0Gs7oOkkOTZMojley6m+hLWVkt6ck53
-/VUpWEhU6f2mfM7e+7kCDQRPQQWAARAA1qCGXE/KwiTLxXQKBbba5E9OyPn6xKST
-69bFrTBS12HiV6bxMGcFdurxREU5PQUNJqY1puQ1nbSYfd94xbBdA6fPdSujp/nB
-18GWZuirfMKiDyFFfsqiSTROGnMFhAn9CnsKwDRzg6Y5G20HOr/Y9U4jnWycCKNh
-nZzwB4NlvtxxZ97fCU4Sz1HhfNm0g0HIh4+b39zHaKC1TRK589feMsJmslSWrsVj
-BdyrSzwbUSt2mjiHY6jJDYFx234na6jneeFVxNNlWSpkbwbtbwBp7CFam8rpnpnv
-6NFOT1Hjztv0VQP5cOV20oNxnUuNqCYFCXfj7UHSFrjaeDW8U51pTDQ5sO4CcEZg
-J8kYW1wfGsFOq7soFl5beK0RYgJKuWUGyNaXn8c/aj7YKeXFuQYoAAHUzBwYaoNr
-GAhZ72CEMAeGxsWbH9nkUeHkfiHp3scos22MAET2yif5aLar72Y3y3PYC+Or756T
-JrkePflB18rGY1e9eNRUmVXumMQKXaFmGiQfQ7fWiYN8jh2ZtFEWvllL5WRfNZmW
-wAs2b2q8aMM6VUuMJRgdk6iv0cscAMLwxWaDXldOF3WVxooQoWOFQhbVQfctPaCH
-mp+EcvVkcsIEiLC2or7B1gUqdGXTSKeGvJUsAN/jTG7etmtNNfqiKPlnhAspA6Oz
-UyKl3xRcPkMAEQEAAYkCPAQYAQoAJgIbDBYhBHV/q3ztGBSuFbSDbl+wJ0dCA0VM
-BQJoTb5aBQka7exaAAoJEF+wJ0dCA0VMdb0P/2bvqk6XPYyZYJ6sglYUHrf4Gf1/
-plKfH1fSHRJ8kQe2M5bUoW3Qjn16+53DX62usUOSiRC/4icsYeZGqIgfpy18Ud2Z
-XYRBWEIMYJwiXFPeyFkuMSd5zOKmrS0g5V+g9w6O+N8U+1qba/C/hv0Js32h8Ssp
-fLpyoF6BL6fmBCObl10bX4Ztwug4sqgWCirX0ewjPNbvGUyjA2M331xyIktGBiKR
-s5a+4PpZbumLBffwh/CVT3fbe7UwveGAtsdEHaJ/ZuTKV1evs+IebBVm2YuTvTph
-BmhVTn4d64LVfTF9XmE1FUthx63i7+MY2fgnaGvjqxqj064Bl5zh73rdEoTQ4npH
-MvpkiNDsRj/MtdMrTing1nHSi/FT/i+/Y0RjQNcO0ChJWKbDrci8AAVeFnYoxf6V
-x5jCpQhdXUb341ej1yNDThU7ZOn3EIBVPsb4HZHtiHYH/ZFghbDB8s6j+AhIXfcJ
-kjju9gYr48XUdob6opyLjfOViaZFmCHvZsgjjrs2L1PcAioZaiUNxXTtEjTxJFfW
-hLD4SmM0LwgmsZAzNR+Uz9fxooClHcHVikGXuObSwjYvzDTpvImeCuEv36fXNs2S
-4ww9qI9JcJT2xXPYKpjLXS/zmComeBdErrxl7F8mtMgBBzXIFvxX449A0Q6y4gXZ
-xtFBUi9ejm9wR9Yb
-=YGNx
+nSXN0A+0dlYkvQ0qD/nV2eJ+rM0tQgW6IBWaeI1pei+20zugVas5qLeFsWR3T6TC
+wHwEGAEIACYWIQRpEA/7kOh6Mg34ZDy+w5AJ6DIaYQUCWlTakQIbDAUJA8JnAAAK
+CRC+w5AJ6DIaYeFcB/904FKLBifM09i+JQoNwOYHfbf2CXhqYX60D7kiRvA+QV9d
+lzm5EloqwanOcviOAR99PytHYHK8ypLMDofWg5k+dY1rnSpn6MC2ljYpnzHdaLdB
+5EpFh8mp3BTP4az3kq3i0tx6m0Nwj4uHkhI/9YUZxklyzoAkkarBj8CzDL7YxKJZ
+XDvI6EmwKGhk2TaUhW3pIVkxzFx0Nd+XeQYZLUqCGwdsMAeYVC/Cuhz8bDkuazMx
+NZ//B+9TZXHVSKxaGfkwwpbuzYD9fOGothesPkqWAEbsUZNXZhcllE2RwOeD/VYP
+zITQb4i3kdTMS4en2UAX33gPIjw1dkKVo5RBqknKxsFNBE9BBYABEADjHjJGXDOh
+XsPtLtPVryZeNW1Q2M4NfE8uWfDYtQixAQ8eUc4YejqwFcPS/ykGIPvyyTzT8w2V
+UQzcJGDaaKUk5bk1SAiZpycWqS37ibS7pkx0/2XZBn7av395ZfjelzD6tIW3n1GC
+UZdpNvg4uL6F8ZAA7S9jYnJqakRx5A/0slp6oe4yWjfeo41+zJka/KFWh1uNC258
+4TANyxg0xCY9+vdPx1mFxR+a8itquFxa7eEcgwAKc46otzpcStZ3w6H3uDueZhp+
+/hyhKMS9d0I6LCy6tVwZal32uuDHGppTSskexUP7w2hJMjINFksPTUY1C2bfq1Ly
+uCD5fKEBehg6rfwvHD1Xj6Q/N46W0VhmzvTHpIke0TMGAb4cfKq/RpM3DO35spAs
+/UxvzX7/lUnpPtxeW/GVhQ2TgikLvhx5eSAnGGwknWSrfnfryNN2HHYyunG4JKP5
+tbx8zoUdlPd9M1QzHEnsG5Vq7rrCLXTyUHdLlnhLvlj49IA7zaprIHwUNGvwBjmq
+bAjW436ypRjbKm/eCEAJRgOhhjRI4pu27mth09x2AYpcb5yKVTk9oVmSQ3A9kF8A
+ZMQV6j4FA8CEsbAykXL2ejYncxIoX+C5Pg0gzSCscR6tf/lj8K3fQsguD+b+vSg/
+UVpF0IcZEnoSVoBUSfjyzkkedWCg5UgfXQARAQABzTBNaWNoYWVsIELDvHNjaCAo
+R2l0IHRhZyBzaWduaW5nIGtleSkgPG1AYnVlcy5jaD7CwZUEEwEKAD8CGwMGCwkI
+BwMCBhUIAgkKCwQWAgMBAh4BAheAFiEEdX+rfO0YFK4VtINuX7AnR0IDRUwFAmou
+9wwFCRzPJQwACgkQX7AnR0IDRUwx/Q//X/dCbVHOcPbWc9iG3i0ZPXrpw00y+/y0
+Rl2UKymX2/Q4Edcoof2UboY+NnoyEXPVwA/wDFNqFJPkVxOl1GtGPRNVBUmW/tiT
+kBx0LESG2xnrV/nW1PfioKDCS8N6FWOc7Cl6w5aNrEDuH5cWGQWAptTctl959ksx
+PPL0wcMLQBmUQdUOGcAxebmFW57wajGVIHz9xzk9sYbL16jyDwHGim+xNJefQE5G
+8chEGHMj/3ckBnaRNj1bVo4a4USEztmrZ2LiEG3oIAH5fICLLhwkdhbqD3W+7JL9
+hU0kazmK1C5JQxTyxj5x1nMce9EPzlijY9qiNxsnFgokQav9Ymd80br4eA5lJUrI
+Tzpxl7ipQsTJZRnwsHbty4ItzoCUfnJRXbsqGW5U0ZZ0Rn5Ksz30DpMiTZuOj5Tl
+3M4U97IuxeOpP7pQF7SaUg+c4Q26MZpEshJ6RVAhPkNIYW5/19g8IzHeZPJzlu9F
+MfVVLaEsRKQVtuQfz1POSLO3rF35SuhIuxQLu/Eu/tdoRgMt5EYfoLhvvrmUKQg/
+UCXHRwdvJ5ODCc9bLrC8jHZ0pRZZRB7Qfm72wL7vR3hlZjAng/KRf3qdSrCwxpnE
+/agkkbDDFzIcjN+vtWRhyZWMhfo7u4sIrw9zmKNw0tcrEjni6KO6VkM9zNonekZl
+FGF02Pe5MP3CwZUEEwEKAD8CGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAFiEE
+dX+rfO0YFK4VtINuX7AnR0IDRUwFAmhNvjkFCRrt7DUACgkQX7AnR0IDRUwZOhAA
+ntw5GbRtMmtCMwk9+oPHWuTeHro05p5Jqb6rzt5ZOysttSyX8dAqSsehHmPFnpyl
+xnbTYbTEpVcAAzKWA5oVSlMu6KI/yjYJCRLNVn0OCcZo0vbPYZi0eVcnjv02ca7R
+26J+Rn72lGVGArYNOZ5esMaXb7RBQc9L4MfTSKgS5+C/RJgwkJnEXe0Zkn5Gy1q7
++ng8wuIQ0+/dIx//G5diA289wK3oIUvDTEqhUZnOCCilYr3aXcKe/80YSp/WqSlz
+ZKc3Ttn7+nIbi2nV1+4NMr9FIzHShf1mnvb7Yqy5gIVjyqQqA6HYCZq2ABVwcsoH
+tZ5rjk0d6jPC6KsrdIZgPJQDzSS7IOIbBpT0rrG0d7ab3A+grRfAqijXUIgLSpJA
+GcCoo8zAtwgVUqdvhQtUO9W1zrLLvMATPn8NA/18UuYmWVO3w/KtVfKb8IPjYVBc
+ROcotvyCp9LwoUxt6zCzEHA2mLQKM6W7cER1VEWxrYmdZG0MfzDfhVMi/U+hO+wP
+nSR0bkoFZ2NlkIMQJ1UtPBhiEwrjv7pDqbsV4YhYmdkIaBc2hghvgCffcEYCoJ6T
+ATN1XVYvgw0gpdByeMS6+56E7VYIlWOd7JrdFI6IOltRid+nZz+WQqcIVfg1SpWF
+Tx7Xd55N6PTLcs6akmc/toHFq0H26Ttp0va4o00a6B/CwZUEEwEKAD8CGwMGCwkI
+BwMCBhUIAgkKCwQWAgMBAh4BAheAFiEEdX+rfO0YFK4VtINuX7AnR0IDRUwFAmYe
+xHUFCRi+8nUACgkQX7AnR0IDRUwzLRAAgrNuuCEzdCZW6Sx5a0D/FNCjbl7RRaFq
+SUJWmwqqHwM5s1nr9LOjFOoQtbMM0iyeuX7r0ZiXLJHV7sKHwktGzjT6tELWeTNK
+3YsbNwsSTSRUjMoLXyQqkBt2MT9y5TYCqEal5Sz28gf0qZx8vIv9KQTRcxaY/+Yb
+XgqWOLsZvs6gXtXUPT/+7S7VKgMhABj/yO7h+ubey/QCKAaYmycgkGnZwUyte+U4
+geRcl0hKw61JCzYEcd6Txkt55Uh6wyQXPTEPTRoTPY/aC+j1fkz3uge5Y3ttPmty
+ZHlVyeUYVxFJt3VDyKdChxjuo1LdC7+s3xmMf+Vp29b+H2hOuOGKfRuRHdnDJYXc
+R/ONirGYpU/3Ds2qz63lAz8rBDJwQHUQeZiWeNyt5LezA2wUDeV+dp5ifg6exBEu
+fDmwnluJoCYzzxrMYjnroF3ztOD2cbKEyT040PzQwtRZ8PvabSnDDiBMV7vaeZ3s
+6mY8X9a4G4q2ualxiWcGHb/mvYc1UkjtiHGX6ye+U6qBZGt5qZYVX/3+JwYHAKWq
+AQ9anJ5+GWL8o2wkstvslRXi6iwMlJ5fWNZrRljnxCo9jOH/nuULigcDhsRFiDK1
+vRyOkIy3wd/KmsfP0Dxh80Fp4NfpmFDMQYg6J1azme0OhvrKiAad6R3nD1gPcXvX
+7sTYPYRx/H7CwZUEEwEKAD8CGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAFiEE
+dX+rfO0YFK4VtINuX7AnR0IDRUwFAmQ7EJ8FCRbbPpsACgkQX7AnR0IDRUwfaQ/+
+I/aIXOPrLZJKRbtYBNVDCXwMD6RY8YJlN8hMke/hr7Xx9CQA/QzCVpLml534tJ6K
+mp1lefd5nQ6xk6qC5ztuybXtVifLv0jp7noetNzI/v4YGzeEOkqnJqK+pI3HHC9c
+yol3RXOz7KzPFSjRNqFjlkfig70qaCgzDYZFHpRem0nVWLIfARmT5bZyWy5KQxjd
+mVSE7EbWW5yB1bopx/UlKWROHzVEzUFivzm+R5yEw8UWmns+NgRFV9RyEgTjUueL
+mBqRpzQclS3/4X1N2kWenHHl2LAPUfeDfHfyLYOlKeaX609JoyWtMxC0VpYtjvHG
+OA2xcFPCIJQPdUsXsv8E4rIlsMhp8WLIv2DzfJRlGM4F27Q5n5cFj4gyIxshMHLP
+4tJNK9X3yXjqXLZ7Mg8wna9Qw1vhdYXNdxNTPRBwVEsHtWq743f9Br/sTfCHa69l
+/LW/i2h4ev1xFTVCIIIFaK9e3MvYdgXpmC9ZtXbftocL44oT9E4UG5ksG72oq6Rt
+STXKzOksntCV/H9uljeiZFxnz846UHCpPfVtMp8pxTSnOTru9o6cm4h3x3DVN5lO
+JWPQVI3JihsgeD3F5WbTfk9o3Ex6P1iUviR3WhqcBauVvRcOVoElSiK6j1aCT+NT
+OyQI40XYqkAw2LHh/B6lRA0YbBrPQmuXuKXigZv/nlTCwZUEEwEKAD8CGwMGCwkI
+BwMCBhUIAgkKCwQWAgMBAh4BAheAFiEEdX+rfO0YFK4VtINuX7AnR0IDRUwFAmBk
+oC8FCRMEzisACgkQX7AnR0IDRUy92xAAjTjls5csQh6NsHffZhSj3O/i+JiX5Ab+
+3lWShCmuiMSB4PiFKkwtI3mJhDd9FpMDPwooUKmTp9i5GwiLFKZEL/YyQTRhwThp
+VdMN6ykdkE4txdX5iYi968NeLvMnPNewZVOqna6WAm3jK4mLf9UCKzbjrdn5LaM+
+DUwOysbaicE9QgieriJJ6U3IIdPVtQJjbesxfdE9z771WijhV5DXONhzC5m3dx5F
+AVWXke4EFwrPr6UgSmCs71Zj2/nsZ08TM3uzU4TSBR0yf1xVeIFO3C17Zb80j+Da
+9/ilfITOKVZsj5XtzxP0WY1MVRAk9TsjQH4lMd1lUFsvtkaARH2g4dQUKxKXes35
+qyWULvE7nE/ISOi7MnQ/hCuBUKCGze7utC13FkRb35TNFGJYVkCJMULI2cacPn1G
+tJKVUkVBKP4MrzwC5sd+57/8+sRnyKwbkTLLjs5lolH9j3GGbllIKqpx81Od4akE
+3j0ZJmbRYPj2u2Py7jyIDfYQBuBHgL5FRW86LQpfkvQPDemRCmsy7pIAzyD2BxGp
+P4VvhkgTnlniGOM80eDjmQH84kAjq2PK/UCAF7O072j2Vr7b3aa7wxrzZJC2TClb
+nzzEBK6q137F3tAtI/+jImzehxw2DGjpPrj6MDIR2kwaHy+PThlUxT2x5jHc3MR4
+Gh/X2qLiGbXCwZUEEwEKAD8CGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAFiEE
+dX+rfO0YFK4VtINuX7AnR0IDRUwFAl5RbF0FCRDxml0ACgkQX7AnR0IDRUz/5w//
+a2+VeXoLLQUz9Q5K0LJ7RABbE2Nhhz9YiTFfpf5aFk52Ht4gaLgCMz4+x0NQyLs1
+7p+6122LKPAvzaIx/E7KBfEC8XGM4tHTgP+AghM30fOrO8bFk01T1CFlc0GHGNjm
+rR+Ypf/t5cdZdBKNheUe7Ze2Jtf17Pm728BVCrANWaFUlQtr4M5k+dJ+Okr6KlST
+lf+Nwm79ZiGIDqvfj9rUMtxVdF1myX3UARR+Bz3p76WQ7KdpthCPGVrRLD6VX+Wa
+807VtYvBO0W3fb4P7+wKcj2wC0fX3F9pLudPPDjRsPdgWCHumtH5AUyzTv1tDKc8
+hAXMyK3dZb28Uyj+DvWgDBWwb7OzWDZFJzegoO8h7rCo6tBWo32zVgjOuURq2stp
+lsjKJPu+sVqWczAQqT0Kk8CtMrEanDsoXN5txhbuXBFeSokcM7P0s2+UtfmrU/Aq
+EDcTKEs6m1vQmme0qThhGWA3Xto5GLNHtXh8HWfLEMuPc1wv4ZcnmcDA7YlaXCyS
+gLtlfhkQKj9ZUsvRUpIm+Wq/eZf4nrG5g4VkZDX7KfFAdUcii4N1inYf2Nbx6SWO
+Uu8iE7Qp0u4ZsPoMv5DguP+3YOSB0rCY4I2u/ZZtddNze+jxyIvoreHY7o9drIrD
+R2hUXgEi/XCC0IMQGlFkpVl1qtdZJ4OE7WrvthSa+xXCwZUEEwECAD8CGwMGCwkI
+BwMCBhUIAgkKCwQWAgMBAh4BAheAFiEEdX+rfO0YFK4VtINuX7AnR0IDRUwFAlx6
+rrYFCQ8a3LIACgkQX7AnR0IDRUyCCQ/9HyJkMcmTjC2AIQrfH6/uakawQ6H9wE8A
+9I9lgWA0+7lYR3dmmTHwsomSr55z8laB5p/HNtETC11uw5w6aZOj0JKU+T53TBR0
+nAwJceBa8wp4mxj7+g3Ls7P0NytYUgrmDjXLVcc2ojJdmbQnl75/YPur1yGbKniQ
+zRzXzgThJLG1/yaC8WrwDgLw7kaEX5+Q6sibvgj+sblk1SBvxUsBRVKXOCQ+iC9Q
+SfBYc1l5xLVxQZYT1R3gJ+xWYWnOEnVT+fM7je/qJQD0CflJCupSYMdACUuA2jFE
+of3odXfgSwdEpZhpccdBT1zASz3YDht6nBG8rMgX+l/wgdtvpXcOmPaY5qXZ4wN/
+Ok+YZowax/FPYdmw8oQp4XWm8lT/bKbIIw6qC9t70qJ9+anZAtGZAIWNRRM3JB2C
+B1T++7SWGYhRZrD+7WHa3jFfj+Ee1+5yjkmWb9GF59kGPmgaxBrjLTHh7qnrtHKG
+QTYTLCvUpYo4j7mfQZGfNBZ1gOV/fMqxLFRKSTavQTd1MssiDJmlisYoyzSkpr93
+FTmy9hczod5r1UTwXDX9Kv3xOHFhlG646Pohfj48LJeQ56/K2FFVe825vuq/IcTa
+gs6groCH/Ms1A/Xw86/5HOkU5A6YZsYTvsp86REgtptVpkANVWxJHmy5WWjm2qlJ
+cXt/Gg+QkDLCwZUEEwECAD8CGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAFiEE
+dX+rfO0YFK4VtINuX7AnR0IDRUwFAlqa8A8FCQ07Hg8ACgkQX7AnR0IDRUzkvA/7
+BNtNaupkDBQd1iqaI5+3uyMlD/Sj1vQSXzPY06vrngupLRHGW6DE68R98v5SgUzx
+VLfuZR4tP/ExnDYgWl3hk4V6SXAR/jq8Nzf0A9CNnkaGfsb+uOGrhAEJ1T12PokZ
+5EdVHH4E/Il7sOTf1nHTxfgoeNIjXPp2kqRcKs/VjaiBKvG2R7OGm60KcgK1EgdM
++1sZHpgwajqOEMfCoLYUZ8G2TG6rZfaJt6ubw3xId8tIZ0IPe9U5ab9YCPfPED8h
+y3pb/eGZgYqqnnlw28KEL05BkpyKukubqafOofcTzHX2svCPPzr5qw/7/qfsy2SP
+pqTwe4Tz3kA/pxoA+p7n2v7AaBodHemoOdD4RTKEOU8oNWuky1+l+xUn1cb2pbNh
+zoedR2TVpvR7qa4NGMFihak9dRoKc4vayUxOh5RP4VL9N/qt9ok9a3aTiLftRZxK
+IpHcYU7r3MhTTSsk1Bik/3N65VjnL9hN25F1q2N4ti2M612diNetuPXzKqMtmL//
+rZln4dT4PE417G5kV4LxWX2BbTuNWbsZ+2WmJWi2EDmqQEmTgHRPbvKanZKT/08W
+kgZOL++L298KUTmJmv4t/EjhRtA38Ru/Q3L8tFsAwG5FmeZUUM5ZlFOI70oc3CFA
+AzajUSjC7RBiWmxzUYcg+zd1K0se0z6JXXgqWa/L9vzCwX4EEwECACgCGwMGCwkI
+BwMCBhUIAgkKCwQWAgMBAh4BAheABQJW2Hu3BQkLWd00AAoJEF+wJ0dCA0VMsGgQ
+AMTEnapuYUAtl60upddiubZfvMCJQ969Hp4rWRRnsMuYlKC3msA2WTdPHz66sPHK
+lkhG+aaFek4QRniqQiGGnP5ttZAZ55moavfPLoItkpCJoqfVi+W8IPucZRPs64Ca
+TBkUnbUCNCLtQH69WoeK7cqZZj2Sek/wLwRi2ib8EWKLoZakL6Sn0RsJEoR0Bhyo
+D/hX11GqDUtk+n3luGrW4398K1BMVn0q/x4BF4Ew9Za+iRcSZhtGZ7c3G6+Yg0qC
+/DfU5kU0KdzwWzkXDreLXQZcE/zfvobvyHsnRhEfHJcda7YKASw/5Fsqq9BDuYqp
+VEO3rVhyp+1BfkYpvgusWdNoFnUmx3eZJRmtzJCSo2C64ww1GZCIFI45mIwgefmZ
+Hmig8xkd8AdJBgovqHzenfNTz01SwlXrySZWEuaJeKECZp2qWqL1nOrF1D822kek
+Lv1VAS8oFdb8Bfkj+pQFtiOxzECat8vhuJMBrGK6CpcONaY1yXG93UNlrAL/Ducc
+KaIIXc5txW9vOedHvRUx6OUHCxAtyL++dYYJ3kvibK9BIcPxFtRq37GVgCtlEQqf
+6hBLqbyeZGaDTPC3aM2dBVTaS6K/vjgfvWy9LkV61lWFzZMxfSOXfXV1JCCHknhp
+cyPUosQTwnORF+K9h88FzXYbvya8pHi1Y5sMoUQRCLphwsF+BBMBAgAoAhsDBgsJ
+CAcDAgYVCAIJCgsEFgIDAQIeAQIXgAUCVPc0bgUJB5diZgAKCRBfsCdHQgNFTJ32
+EADUEIshtBlCKNa/m3PzNToxeH0aJEJuSgsS8bB9Vl1dWxGiwFVMYdb8MU3eenUN
+6XPrVHAA6wG2ZV40G4yUz+RgjqiQSu11UfILPziToL5HCTS/eZ1fWZHFlbt70AqS
+OkX7Wm6l5+jE5lgI6ryzMiDgwLvhy9FUXT0C/QYJR+9P67goISRH2ZlgXzs314UZ
+//Gnfdbg7kc1AY09OY6eLR6SGTHviaJP6QzZKrkcrYyMLM8k+aHuGBtoz1ZyechD
+Q/4QGw0DZAEhWjM4KKuGISyJ5QfEZqKr9FZ5KZOl30l72UiIb9ba1RyYPAF5XW3o
+RosSQU4JrvUf49Ovkp91e5h7GQAGE1yQ15wckRDjEWINGas/OjZ9Nuw5NIbadAuS
+MuappeIjDHK4zfpJjtiv6ys5dG5wIl1pRETSZDLWWikrME7s/NWAnT8TYM3PPz/o
+OwdK8gn9IDaQlTzrV+LWo1sEIUYub9uxurpblPXCwdbbbV3f03yYeMSzwbe1w9zX
+ipxQrxk0vwcwmSldEse8mJp+K/8tUew3nN9jSSH5nVExs71lqjiVH/jZFg+KGd0y
+V05qQczAEllAf8Ltvl2bSQnBxF5mbe7qcg87Ec1YN7yeKIgRyK3cKT4qH2Z3nBaB
+eBnbko5PqpFYE2kcvlA0VTWWeZkAFxxh6qZJ2HJr2fWe2MLBfgQTAQIAKAIbAwYL
+CQgHAwIGFQgCCQoLBBYCAwECHgECF4AFAlMTcioFCQWzoCUACgkQX7AnR0IDRUxf
+ahAAw1h+ruXtkrpHzQkyaIHkiY+ad6RC0Ki++psftclfIoIf7jH6noziFG8hFnvy
+kEeaTv5S7O/dpBtGeYH2L+UiBJdmOK2kSepO9sTXpLwtWj1Z7KV/WJDjIkbik2Ee
+MLnaajjR34N3aYQZOI88tYQiqwRtP3i23lmsyoYXriONMEmmOdNozwDUP78JGvfe
+iWcSpX4t9YUgf9Nfs4+wopikJp2oAKW7r/o/5vb/LzbSStLIHXWj5CeCSM55rpQw
+lFJtOtA2HxOEKZ07gfP5AyfnO9cgTIjRYBhh11WsM5+RurhbCh8FeRgW6Fxyg9VE
+asRUwyzF7RjPEwjaddKQwBU8SQfmPv61I7+XUGuSkGOgOiCdEmmz42dJI4EG1D+5
++F17uFBcxGqWlabsGQcErPMuBmjhjjSttrRNL4brj1pki9YpoFCcxLnE/d0N4HBL
+CizP+hxLefXQ7BZ2LDkT5C19kpPc92TUJY12XRqFP6DnjvquJ4A5ymF+98InWLyY
+pI2Kg1ZQlrEQjLLYfqJoGr/J3Byk8L2fG3bemJx7+uG4o4ZPYFtiH2/JJJj2a9Cq
+dJua7rY9RGJevd5dC4pgDlwCmYhp84ZKJTTCRrf1WPvEYK7fJjb5tOGSn/n/AgjJ
+gnoyFYQuF9GV3R8DbbTb6vVNjVS3Q+s1uXJWs+Bc/l46qF/CwX4EEwECACgCGwMG
+CwkIBwMCBhUIAgkKCwQWAgMBAh4BAheABQJRMRV5BQkD0UNyAAoJEF+wJ0dCA0VM
+8uUP/1Mvg9Tg9klod+CtfM9kaHgIWXH9qfCB81l/xaKX/g00TK3GEyZlFkNpc/SF
+9IlMkWIPkbicBrAzWrmEbpBqGo9t5F57QilHAvPGwm4DEeeqMWVJZvrewBZ4vzZN
+FOo4YcPMI2yFWtTs0IJibiYUYulUYjlYwa8yCc0hXkmMiQq1gnKQFCa+y0/eKigi
+xrqB3SGZtpexLrzthnIJ07o9NTLC3jccZhv+W+UT72AhnWGmvs2sInTnUcWmMz2B
+PoJqOAPVHMBZNbqbMuabnN5uERIe4E9vMeHpcEKNjc3bw6h/CROXqH8XrGBJ6wz0
+aj9/PirKHtNIGHcOMip694rvJpbh0h553kXGVSiIsnCqRNNTjcGTB9Owy2uru1cV
+Mol8amaiCDDKnDLVzRTkskwNle6+DG0fdH7aXu5qo/483J6869y/d58DF/qSKUct
+oIo/5uDKGW1P8oQb9xQ5O0QCOAfkyuRAHIfF6dGEIxlxlCIvCWwCdBvmre1zO/qw
+CjesnaSHde/ECUeYTmlgrraNPPgpbismWdcUGWcMcB0clhXNhLbpPMg/kbBAC3aY
+fFE4Jq5MRCCPi5yimT7Vg8xzDamDZtjkigeYncfUzickzfBE5owm6KUyWQwTmN7Z
+hWKREZl4UkaJvSb5yXmIq2t/DERwy8CjFIy2PdOFD+7Ife6JwsF+BBMBAgAoBQJP
+QQWAAhsDBQkB4TOABgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAAKCRBfsCdHQgNF
+TOtKEADK/u8XdrHgTdNLiMzlD8dqS71OU1I9zZ/SRab2iSzNwGFRlQpx7UMCLQAx
+Wp20S3Ts/DyGNIZ1kUvAkNUAFnh098FuvGPqk7U+FSpiYsKPUJYUzMWC7Spnm4cF
+n91s+dMTSS4WMVaCm+Rfh6/SvUNeQHQh+QFkFaPZuJCM3XY6CS/CKLfAkV4ONVWK
+IdtKeQPWrlKp3aUJ7n+42zcjbBqbL6PXkG8sO85EyvJ6TENN++ABanYKk5QV5ezs
+cVWcQqlQRWJ0W/XeJfcK2cqjr4p6oDY9koJSni9sPy/5X0szb5aVrtskWn5Su7IZ
+EG3fLSqYXL5B5mXeiz4TexilecE0E7cx6B+yadtTNPJvjbIoXBFtkXAkOJY+CF9E
+lwwJ1TjbTh7s69IalPiVdfx+W44w6XV7MQ/ujN57bOOYXO99QtlnG646EnghYP1D
+1qbDBEtRqducTSD9LW5OWaPLehCnSWCQkF6ovoJz0KX7siea2OU0TffvS5S580Qf
+omTa272Smvpj55TeUghErASiEBi+xjHDFS3k2jAddw8Ox9brZ3HQS0NHvOH+mc73
+xc5PRBDmeVlF7aQqHd86hW+saAn0gbC53296vOjaCh1/QHk7iHchR6pGFcgwLlio
+o3yYUJWzpl+FDxAA2fzBMEPSkm9wWy0NCJgbo/xBvI8M1lveX8LBXAQQAQIABgUC
+T0EHLgAKCRD1Mr4dkI2LDnZGD/46jVaA6hYzlmvjbzYmPfZvQt3FhDk4GAcGsvtd
+ZK40Vpj3cTq/KBNrGnSNx12Uy9n7geK3XKtWibVrtB3E5+6keEs9tDR5Mxx1CEMN
+zBZLyMeumeSEqNcepGo6opI0xIYQwtoiJGRNphwBOdcRORhbSq65Bj5dd9ZaJIYD
+ILQEcmyxpOQ1wxgLC/EbPt/vSH4jd9f8FGoG9GyAmc1TYSNdiFLFM+5gFbX6ZxNK
+s2nAR4DHNPrnP3eEX9ecZXjBOxPafHPS0L4yEiawQ/VqUhlPT2p//bgHSkkOSQm3
+PIbkmlco6ZwaZ9ywubMlngjRM6ND4qGEqEFx8GLqqrlVK6yi0PsQ61E/CXJmS5kX
+sy9TqjcHGcRl9krKLmC8pPJQoQwUHIHPrHUte0X8vYLRwDpasHkTY4jNC+yqKtlY
+7Vt3DCSfNPH78AfQ2rrvKLQ96Nu12XvUAOsv7If6KscI+zRo90GfUKwC37ZTDWHt
+ExnCx7TvKdF208Uq5oUx53HOm8KUj4glJSas4rjnOZmYLeolJFYjkhSyQRhHBu5M
+KWzUV2zOPFWk7xWurykGZiWyquoKgBcdviNMHQauyU4rWy3JOMAl2xeGnLP+f25r
+g4J7F+C4NDQjRWGWvFXvQ5tvY6tUvT8DKDcNgtcnijiHBrw8eeIWw+tQWghVaEgQ
+sWpzL80wTWljaGFlbCBCw7xzY2ggKFJlbGVhc2Ugc2lnbmluZyBrZXkpIDxtQGJ1
+ZXMuY2g+wsGVBBMBCgA/AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBHV/
+q3ztGBSuFbSDbl+wJ0dCA0VMBQJqLvcMBQkczyUMAAoJEF+wJ0dCA0VMzEMP/ROJ
+n6j9QR/vEbKG/6svXPrntjQZ2liVlDAo2KYA7rRjNPqh4O2rrxYHPznLbwgRUHu6
+wTVacoke+wDqs+HojNiHbhFAYO40PPabRfrURpwW2h+enJDqKcktEAnS3mFUk0Za
+Of0TS+Qcn/SZ5bCGPxhY25Wf/4gp/vRSM4Itn7a7Kf8PU9O7ob1m5gWOML4upODl
+0ZTIKR5lJUknjwtfyzmwjMDBSTzOdQNLjAXtZGC77ZWMo7AKGLC4VPZKnlCussi6
+IaIxcZpqofRUJn+6gJHHscXUmzNlphUaK/73tU7mK+ESK33biBCEelRWeXRZMvJ8
+Igf4RTpQhRqoLsvQLPIx8T1uTDrFhYyK+J8J3k1BmB2Aps3r4tj7qLZfkVVcI1bg
+br8AxPcXvTaKRkMDMDlzuSOrbv9rzNvKsktCichGkXYFcsAM3uqgkNntyK7Sa4hc
+rovg8ca//VVDIMElilDgfI9iOofxk2CpJ6mFXagtAo39Sv19V7MCYs7WX+deYbTy
+Slf6Fu3qVApoUGy3zOCDyN08wDA8erTuUGJbxxc1m3S6zt+WcM2rASWI4vm6AcFf
+B1MqmZXtuolMaYVaowtSUvCItBHrOGqE1OY/GdFfHwaa3ximhIHlSLNhCwuJOGtd
+HLPdRed7ly4JVmRt3u7zKzpD1BZJtwuLUHNWqTJYwsGVBBMBCgA/AhsDBgsJCAcD
+AgYVCAIJCgsEFgIDAQIeAQIXgBYhBHV/q3ztGBSuFbSDbl+wJ0dCA0VMBQJoTb41
+BQka7ew1AAoJEF+wJ0dCA0VMji0P/31ToWefMGnFwn8YdUCpO4XpjQltAivcgnB8
+oNusOLvA1gPl+id4uRmhltNKJtAk5pFKzo7tyURTuKn8+lpQfxBzODYjm3s6bjG2
+h8/DBl3SIm39uJw5fAbpHoh2wvXIwS2GI4SAANoJawuKURYuk1qVZ9P4QkdfQ5Eh
+UrefRwVfO/MJPNBiXh2xi0twq55/xBBbecrRhVHK4B/A4pUSYAgiv//B3xwJeII6
+vqEGK7MeeMK6gjm29eoX52qe06QupdsjXITGjkYpbkXcByknHX5c3XRg16DHJO0d
+B9PkFr6ihA8T2I8R1fJOqTWruRLRerZII1YgwNi1MLWUfxiU76uzd6Q0XwdGHR6F
+jTuel5bA+hc7kAI35GljL1bEuqeuHBT/jgxZ00HcnpoXEd5lrEfjR2pwtgt1OERI
+vZAgt0EY7zSVSiod4o5r4bTyJcYBq1xtKeBcKDLl1Z1gE4SvzQOqQ3iLWMfANeAe
+ksITBHbneRPH7YHXPpNf+Xvsm3L+ejn1AfTdyHvdc4fkGcbYXahqICDmItulzZLx
+lywMMTxbZRSP11WsqsnZgSEaj40PgF2prfflvq3AUt3hIqpSIXiNUiyleNjHHV/5
+yXxZ99UfDMFPSeLv8N+CZ31AXtnSiSHoHWve9OblxjvbzWXxjYG0lB9KLYMtn22I
+Ped/dYwKwsGVBBMBCgA/AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBHV/
+q3ztGBSuFbSDbl+wJ0dCA0VMBQJmHsR1BQkYvvJ1AAoJEF+wJ0dCA0VM4sAQAL13
+gIJnpsAPKZHvgGEPHBz0ef9FQ/8d2Wn2uNXXBPR7FYjrCAhvzM1HRh6F43ShYeYR
+u4JPzJOKEom+c4JNVkV/IcTli4pbVAzayN/YmFUWpwI/8KwEVZYFwPjrsNIezk6/
+pRM42kPmdm9ssM65lpMu4VZk5y0CNJP+4yU+j47mvTfpDrel/3kRycXuTEmL+pEI
+3UkFS1c/ylDl7Inlu5brURf7j9HGtSPRjyUn/Cb/EoUg1j1Z8FXXGQlH1nHgN+B7
+w1pM5PehTLAdRn9EObg4Tb8vmDWS9AcGgLPpqmpT5uD+oiCmHmsA3EhWvq2EZcem
+HcO6Hgn6Y43lqBrwH5h2Pv0i3/V1b95JOhvdL2nG1pyqyZN5ntHJewoLC4SrGEDp
+6bD/k57lHd4B7yvdmwJ4ichikTqeQqHy1iSVGfg3RkOsBBtAGv2XTvbzGGfkQfGL
+qf/PxNcexXYrevJC55Ok45/bGPJLYeGsimNpWtQy6VD72qXImiX0rbo659aaATgF
+Jrg2eGRTKxG6svzTcdBWQ2tibEvTlZQLWzioESS21SS4Ku9a05Q8lRKKmjHI0+yQ
+JbK45I3uWVBqjSRr9nrinZJJf1HEvqRksgtsNcMMoDODr2IYHaA/cBUk6RxiGyMv
+WTZNdAXn7s78vfLozgsyjQ5Y0YqacrwfySv/nGARwsGVBBMBCgA/AhsDBgsJCAcD
+AgYVCAIJCgsEFgIDAQIeAQIXgBYhBHV/q3ztGBSuFbSDbl+wJ0dCA0VMBQJkOxCb
+BQkW2z6bAAoJEF+wJ0dCA0VMajQP+wSqHOJSX7rGRpWGKqeN0nLcGGOOrAUx7w6h
+Lq+fStFvJCPk9eiLQ+DAg8/P26i5uzE4ipbo5U64NIfE6YhM9xjuZrg387EJuGVh
+ToHLjqBQb9hmUe129sZ1k16A/Zh6YnNg8FmFO3Ja+HkQfCwda4dqdvFLtDUOHNqW
+yvJQ+M44nG1NXQoC1f/9DFO63XP2+2VhfBuTdzByfaJV/M0e4poGHZmeMAKamvQl
+vWs7S7WrOoJ9t8AOH8iWLVjR6oV/c6y0o1WM6BrkvrkWFdlI4YBYfpHg2Qqul7g2
+I6U84vL5j0qruaKyZSmJv5BAaHN2RunK6Ve64mMBC3E9p7ySaGx3K9AH4STkcwTJ
+aJShSg+fLqj73NqkqRYX7GUxRk1AWzzM08vFszDrvH/m8kherbJFCb+evVPaf71z
+yIOh5XCiKNtBWTWGpvgQFqKLk2N9UJkGOaN55wYcpXJkzPoP+kSiBtIPAC52Jmb2
+6jf7SAvVUD1sBJKOclitNmH900PEKsXRrnVQv/ufUT5Ydthj6mKDIM4+2zZh64bI
+SniAvVRElm22YhU9jrxn78D4r2QDGfZqFJr6x3WC85uG0qXszzBEkGneBnuqzWCf
+MIqa0Slja31e28cl2R61bcDr+4JnY/VaNPr9B3j5hH02H+fbI+tnrThLCqGe0CQE
+zP93ySHswsGVBBMBCgA/AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBHV/
+q3ztGBSuFbSDbl+wJ0dCA0VMBQJgZKArBQkTBM4rAAoJEF+wJ0dCA0VMMa8P+wWQ
+g/LdPX17c7S9Y3xx/ObkcDJy+00jqDfKssm+v58o8oOQWO5rBAQffH3AI7TKUiF8
+OlCuB5ajK31TfJbfGu3RupAeXymRoyswwEkF+sclYxpD3SFFJmBzHMvw4OIdfhZq
+T8M6JnVX3owHzKCurC+/HqLRnsnis6302vCb3NtpykOYLLa6cubjr4WHGmcZY+qB
+4U3uAbyvxnMALa36hSbMJ5WpSq4095pMK7HUZr9QxeL0y75cvqNGSWYbxqFAC1fn
+HBsorH939g6+irjs6WJxcuUnYtzf5nINfucXefLMFXgjW5knHazf2Mu2jTY6jgq4
+oYe44yeDabdijS1jjDYtTf6KS1jcGSrYzyC3b8oMgRTehRiradibqAELYLpTeIMT
+OwAUJ6AYMAVCiUrzkh5XGlSGZJJIz263uK45bbNN3DbMzB4r0Ajlg0bH7hAEB+Lp
+j1J9LY74PfUSXFt1aTSAPFE8aBtjrgNUnp9aiHiF/zIsWVoFqPPColYz2OA38iFU
+DiYuPss3rh9A9QEY9+9tEpfihyyx388W+uqS+UZJaXP7axFlRUUMs1Y+ZWInbnC8
+MwOSsHgrvGAVxTyrqHVwZpgawACxYcUNove/QDfcqYl2VfIxv61CxgB8r+7BZQ1Z
+geUw48zFfjD/SoIP1SXNAR61MiVxYUSQ/sKTNXCnwsGVBBMBCgA/AhsDBgsJCAcD
+AgYVCAIJCgsEFgIDAQIeAQIXgBYhBHV/q3ztGBSuFbSDbl+wJ0dCA0VMBQJeUWxd
+BQkQ8ZpdAAoJEF+wJ0dCA0VMm80P/2tMPKyiWBa0pF/Wt6XpmW8n+fhS6szQZJhL
+9Tb5mTgvtYOYTuP51ckkmWrvn0d6SIwGvl60SWLEy24/BQa6Yuty2FvHVgF7wTaK
+gWp8TF1EvbuoyZ6mlyue1L/wnFLrNSok1+bnxBdAPwhLD9FNZtJGuGlI7jqCmJ2U
+Xl/mR0VyN3aqcc0rQKPMwSZTTmBPcAB+8w3QyLNdjtxJ3LEy/NhjBiPKuEN3S8zR
+Qmky8PfY/AixuQC7fyp5PXQPj4lNfTFMWsrZbJNIBj4Zmh1JmXiUv0LNYhUldQq3
+24T+Uhr7UBn6PXovka0w94s0X4NbScgxR3HWQsLUjLtFTfeWEg0x3vD7rVoG8BlU
+j9imcDANyTVqu0NSSdWJqogT7BjfVM/yQ/wyI+SHR4FLdwG5azRRNHhk39VFzK/i
+5ec8O6CPsS9Fa4BNUhMaK63No3JTfwYaYe+obHAgCaul10CtDDfAV5k1Ftl5PynM
+OWMgHOOBqZRUpjiiu54qXFCZcWlFdZqhsOgvOzNqzMfIwLwHvX4MAcTJyI45hOtf
+icQWEXjrMhdKIGbebEu380jb01Vs+pp8footZEUA5RJ6XTkW1+mUcFCNrlLLnYIG
+WMRIeWf6eTz1W7w0OxRkjresStgiO9XwHDZzZpETBKHQPCHHILod2TlVbPI4uDoJ
+DzkDwhgDwsGVBBMBAgA/AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBHV/
+q3ztGBSuFbSDbl+wJ0dCA0VMBQJceq6yBQkPGtyyAAoJEF+wJ0dCA0VMkykQAMJS
+chMK+knR7duHTyOwxF7NBhBYo8bIrJdWk8FLhMsOaHrZu5EdFignmUJKrmQryu18
+SCkh7v7bFkdhyO/jZo7yD0A5jSEZLssglCiI7tgXDKmqlfGztNHql1/DEbGPrVB4
+o8ov+umklfBvxthMvvLrxdHIpgYIERUZgXCzA/kOw78tcLw28sgG6bXW4Ieq9pD9
+YL8afPAidf9xTzQOlH5dYk7ZnCd59wq9y6tzbR3g7HaL9HCjwrrNqzh8E1vcDRCW
+NBD5Ol9vovyo0QDx7HGEf6oPQJIhCaRWu48USSVnF+2395v2e92/0H/rdvyOrLvM
+1iT3OBpw1Xnw9VqSGwLoXCFllF4EAuYD+pbX5O/9UUcpB35MqWugQMTO+XfYs7cp
+hV3MOl4s5ap+LLNHIXdIHUexWf3G7Gu4ceNYXDWXYLKWeKdUFJ7RXWbwy1mRx/ql
+1E14NHVCd4aVPube2o11CMQr6NgphKYfVoY04Jn2JfVW1Ilxpv7y5Fti5ZH4QLTP
+9B+lt7JYEf+sf8sxuEKevwLa6oj6K5shB5JDmdHMviuCY8BaTMVmcck6uvKIYFep
+AhAMzhNS6GXUZJtFXukREdOOjZIru0gdOAiM3v/A27+FJRlXUe0IBz1MdZYpv3WM
+dXxx4grfZso88k/K6DoDnRsAOlDfzJ0OTKKLcFVawsGVBBMBAgA/AhsDBgsJCAcD
+AgYVCAIJCgsEFgIDAQIeAQIXgBYhBHV/q3ztGBSuFbSDbl+wJ0dCA0VMBQJamvAP
+BQkNOx4PAAoJEF+wJ0dCA0VMyDkP/jpSMrmaXggA9jg0vDxLY+ig57AZ41gw0Zm3
+4veQvRRx3hBIZI95AH/m0CXS+QvwwEqKB7EkvazXvbkjEsHMuOwRpWvAaGdleAKU
+uxIg7AHw0OWYQdCSXldHWMRHQ/DowR/sQY4I/MgS+CWOMzE8bNqqzy7qGCdfinsd
+BBBCZgLUfcTzieiYW/VTbWGfQaWOl/rKxKDB9UzkB+ux/kLcd8WYT7hSXsaCqDhP
+qTH8JW9Qy/2aqIpoG1syrHGsKG8awEZQXlod8gG0/4lX7D3qs6aT7kp0JKQbNjkl
+HxMI2AuE5WIN4nTfLWNdrTlbhhS/+8byxIPrWzV8dHL/Rgqsa9/Q6N4sVcngwv3R
+3Df2Ih4a/wh8Mr7eWb6wI2cAw2myM72IHDuP1+MEV1+kI2DMftQLVz5PzYpuEKFG
+NLRPe2fMnHFI/Ekvxz5tZOwZp/OpW6Q3t3IEZcN642MXi1P9C/igfQyVDZdIsU/X
+LYQbqw/RcmkgiXoordkuLtANYBr8kXPRr8bZ8uPhtVzZh2SYp8sHPATg0IYrXRYt
+8qIbGepHCDox5XR5p2vA1YpuYo4b3FWFhl0EdEvbIZqg3Hc3TyGMrVMN1BB+UhJ3
+sEoBTZsXd5yTnRUg4nn2X+4vIbGcgmNnxXPTcFQenJuLUZ7fRG3n/lQJJsnyZSKf
+zankBdEowsF+BBMBAgAoAhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAUCVth7
+ugUJC1ndNAAKCRBfsCdHQgNFTElGEACUMAqo/WMc4YdQcHQIeP1rwrQN1fYSXOLh
+57ruoHQX+9/JbDcC4yrBc7ydPV3H3S0Gg1BZR3n1FjUAUSBZQxVxWSsCu5GaapX8
+9pjFePmIZ2zKhpwPE8apc0BpTQRH1MctIJnqglmeJM1VimitkIjU7QFMQOBdRaEL
+ABPTmAuECMkvUUXj8wPb/KCg5Puw+MqESYmke94th3hvvN35Niy5d2btu0NXfSa3
+3fojrkUsHogCAfDLKtJv1LV/VaiCgr3/XCRGre200zYnkdVX4QdJoEYXaxlAdVXF
+dmicxjVhzJO7hEk8NY60op8kMomkbdwMv4XjwE/OcgMTvGFA9wy42gP6R/0qUn99
+/fzj/hHGo8JGkCKpgDcFfIQUkMcvPrBafx2p6Kfu1XFXIOFAd20WydeEHS+BEfwO
+md7yHVMhFbuoXR2KIfDechttdAmBUAnNr2owW4s3Ch7kdNo697aDXjTrv+VDgGk+
+JZaiiUtM+EvFMgJoKCy6pGgZETQ02ECfGaVWzQJkF9ucGtb8TxO0XC/25Ii6SulS
+gXGZ81jYWoava1f5lTnU+PB91LPjhMBvC/GoE69keDvyP7jn2/v/nTdLxeehLXZL
+dGKiQDFVxpws0GghVrzV2pAQSxG3YAqsGI/ZnZQiIIKSFRIu6JzdD4hYWV0GppD+
+AmupV2mmD8LBfgQTAQIAKAIbAwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AFAlT3
+NGsFCQeXYmYACgkQX7AnR0IDRUyb9Q/+Pgq6lAfAhGMKMdIKjAr2Nf9jeMpaQAtk
+Qxcl9PNj74/eyrzU7tTQGHnsjiJyUghL4hLfbFFwhxk6QVe89cH1Xydi6idWy4j0
+kBk5LGByhI8p/3Q6MWN7JqWIphn8TYPbrNJtzgEcp9Y0lCzqbrF9QHMdUtuJMtxT
+N4kL2fPk7JEj2e1Si6FJJla82nqcu9BrKIGfMVJUO70aFlNMc8dhiQoweHvP42Ka
+qcYRYjM5ynEqlAHF7k+y3k/Ixttd+tZGQicQPGoyhnENiYFfVuBMBawZ7x7YSnHW
+1EM25k/MFXA+MOrcNj+ipqy199Py5MtQwuaori8aP9Kkhj1ICqz/6qlmESUgI/nt
+mKrdy9VmCE4at0RApTzf4Et/jhxYssLtofkwtEqaUJ9eU6YSUlkIEVmI7pj9S7LO
+0A9hi1S0M/Htyp1tNJTeBOU8UdodRSqGtTMURoWKes6iAiD96JOoOnNo6YcNJwiS
+81B/i054ArbyTcmyfV2gp8pGdmZsBgx3TKVUsQ579i5YtmZNQhmsxQpmZKAK1LYU
+LkuawAw4GXWUlZNyiO5s0xKUR8YRLmNpN6u7pxc04aYVYeh25iv91yJDt/8i95Yw
+f+QzhZhd8R8G2mWvEdU2JZi+QnarfE7Wo3F9fy/yLFyQYybeDZMN5C1MwPaFjZIS
+pl1iHNkDj0TCwX4EEwECACgCGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheABQJT
+E3IsBQkFs6AlAAoJEF+wJ0dCA0VMz6UP+QG6ipkdOcj8frp/4TE4c4E6BxdrBQB/
+9owLFXVuihmIkVIHP+yjBSy6HBXSaQhjHGmrythMkqMwmZIYZCXj/ehDOsMbycZa
+UExiMr6/koZcHmo4dRM9U22ydHt+q/fYFd3sdI9EeIJINDxal6d/3kRrwZMmvXFU
+iW4Bi75BwUtN/whbVI4nRQmyzLyglq7K+A0zhU9sWYMgABu81sPgcCMAibiai3h+
+ADdNAjlIMVSShR7Cbzfu11gO4a76j14LYK5BqKQhxfODW5G4pJJ2EYheGEdkmqpD
+eebVe6VIJLq7BtJrRvawCmQvfV701sisx924zolP1BZeNXIN0fI8berdnvlpPotG
+Zz9mq/1wDCh8t0deN1BeDGWrBZEPPQW7qbK0JVfV2xF4iCJJ4zc3h64qsdhBYMn8
+J1gYUbBrwHr6vPN1i5iiMF9Ni62YFUU7P7j6GvRavI7Ac1dyOhLWa0HBp2cXgRZS
+EkRFQBCNQ89ABukAYIJPtxxHnrSnyaMVxzAUQSuvD6O9to4+vKUFOChNGeF+3Vir
+/ccvbLhvgyJzSYPlbxPABcJr9yw98cKC10KG0d+3tqweonyWWpbtJ9sccUQMeVT4
+W+9PJVA71AfLco/6lONtWPAisvelIuAsGv+gUvw4+giAdr8oon+roGLbBRfrJxWx
+kxIsV94ApvnRwsF+BBMBAgAoAhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAUC
+UTEVdQUJA9FDcgAKCRBfsCdHQgNFTIPvD/95vcZxAVlptQJfoSMgkeJZCPXZcAoG
+9sPwNIV4laLtvKMRlyYzFcmxBR4J+E+aALxYqFsbR7Nu74FE1exFYFMLtaQojK6T
+r425ST/6YLsMnB8TV91fpnHCgTjDrSkhdiETGbk8Fy2V48JS6ezyVsNtmcga/AFs
+ARCUqQ5/pGY5kDWnjbbNM2hBvJRXZY/uZJomq/BMloUGnTXzUegOSxLuJX5nZXHk
+yTXkhUkI1ANO92WGqNf6GTZ0RJS9xogVsR0ZjBZErPWKjbUPS77tvecos3S5VYhz
+pLGNw7T1crrndRgDn1TFQGtnp+/AdAg5iCLHwON0DMiZ1+IdiRQEZN6RxasF07m4
+ZQsHO0HNWg9WeEQD4Ru4C89lZ4dmH26R7Y417+encsM7H0P9u8WmRgleDxkt8NTV
+bx8TpgBGFQ6gsJXuDBuSHTnloSIu6cU2euFboHXzwukZ3L16k70lXq6W0z188qpt
+D1xGe0v3XR3AhEfs/PAFMzTevhjosrB6xTaOVWeXxC9WMvNpr1r4zj96pFlaBBYK
+q9+99swyn8Rzt+ItcSp+IiNcVJc+gY+ymP3IZrfnX0eHXGgulQGADkr+DJxO9/D4
+MBZPMftMuyU4dQuxJU+AmmxCxSqGYeSk6nWu+2VaaeDfvJlM4ibb2RdKLchFEheN
+PFsQqxYqLuidpcLBfgQTAQIAKAUCT0EP/gIbAwUJAeEzgAYLCQgHAwIGFQgCCQoL
+BBYCAwECHgECF4AACgkQX7AnR0IDRUwTNQ/9Ggw7+5/L3xYgpMp0R3X1XBraneHs
+wmMd3ZMzVpRX3UZRbNyc02Lv+ewWZXAsnKmPD9QK45f9j9RyErfyvzzIXFxPHsbs
+pmXTwkz5QGLhDZsAMoS7bqIL1nldTOO2Cd0e4jGv28dGOTHmik+xcCRUDl3/geB+
+ZU6KqwePHU7qlISNn/wMgheoTXoyvbKRpNFTiIGVA7jhwDRbA01F0RDCbUvFrIGF
+qCVAO+6A/SO6VtmsZhVC6SE1uIs5oW+JUxu/xggbJ6ufA7Bmk7v9TGM5s/Inv/v/
+xlh8ciRC0IIoIbJSRbhnVdz79oDP+eypP0qAcPU+XhYZGJCjPoHqduH6Ao9DagXx
+zKJjY097OHCJAgf++9sm26p42vSVEMhQ0yj+71q9MgJskZ80Djyof/0bW4WTHdfT
+dGFCB9jz/npcm6bgi+30mRujvLoShWNEeLLUHukYT7tRRlzO4uk5IvOi3lltGwcx
+QpsW2yzwu1X7vxlyALkJQsXRdrIxfCp52DGQyLt1sc6LxXyGB/h5pvC2BxJZcg+M
+FveoQtB/MEaOnJUJzFwdlqHeEaZV+UTRWMFYyNPyrA53VO1XnLFQtUASR6YunUHV
+4n3NDYFOHi2yXcuusda1OyvM0UIb17NBrO6DpJDk2TKI5XsupvoS1lZLenJOd/1V
+KVhIVOn9pnzO3vvCwVwEEAECAAYFAk9BEKoACgkQ9TK+HZCNiw7j8xAAwTGaWoD1
+HoP9O1KXI5DdL3DkHrzbZJ4QdPQTJx19G8UmiuGvqm3gVuVKKjvUTmrd5E1pLeUk
+zgTAZluTNqize1uAre28SCetUDzNJBB0KZNbNfCPgGXIIvUFJgSJPPoxnWBSiJY+
++V9hpf+IlUYRmLH98DU/Tufkl8mTCeMEdDOY8z9/zrQE929y3YLVxyZSHsoWbY5Y
++i/H8QUZuA4ICMjDBQ5x8NP1QllKArTqgVt4fm2V8oeLei7ednOFb4SIczlPKNwa
+5Fit5bFIFZphhISARE9C4HTwavJczX622ediOWHIgl/5aLq5MPE9klq4XMU003FO
+Uva6tO7+hqrLeAolKtc2eHKSU+djxdUEm9EGnoF6oYq8YKg/Ho1sU5yRIH0T5XlJ
+jhcXCFRt+S401EAdaKK1Hux1XsdB40OUL5PUj1wqEzzVJh308IsMwBCuNMGfsXI1
+CGlmgYFWIkITfcNASSqJtLro9SvlzQEHZ8KVkhrsuCVmkhdpmAam5KV9oxLAfL8R
+mw9+mSVOPh49k57RRgsBpxUtMEYLjlcoYb5+lyY8dXn371uK5Q34j10RIFuDutlv
+CZv0/hbsLf0wAzYqQtGRopifQU8P3+KOa3MaXfbcOarrn0XEJSd/zckxmg7f2KR+
+kZvWTWbMfGFTivJRHFA7vINDvs/sFjegeqzOwU0ET0EFgAEQANaghlxPysIky8V0
+CgW22uRPTsj5+sSkk+vWxa0wUtdh4lem8TBnBXbq8URFOT0FDSamNabkNZ20mH3f
+eMWwXQOnz3Uro6f5wdfBlmboq3zCog8hRX7Kokk0ThpzBYQJ/Qp7CsA0c4OmORtt
+Bzq/2PVOI51snAijYZ2c8AeDZb7ccWfe3wlOEs9R4XzZtINByIePm9/cx2igtU0S
+ufPX3jLCZrJUlq7FYwXcq0s8G1Erdpo4h2OoyQ2Bcdt+J2uo53nhVcTTZVkqZG8G
+7W8AaewhWpvK6Z6Z7+jRTk9R487b9FUD+XDldtKDcZ1LjagmBQl34+1B0ha42ng1
+vFOdaUw0ObDuAnBGYCfJGFtcHxrBTqu7KBZeW3itEWICSrllBsjWl5/HP2o+2Cnl
+xbkGKAAB1MwcGGqDaxgIWe9ghDAHhsbFmx/Z5FHh5H4h6d7HKLNtjABE9son+Wi2
+q+9mN8tz2Avjq++ekya5Hj35QdfKxmNXvXjUVJlV7pjECl2hZhokH0O31omDfI4d
+mbRRFr5ZS+VkXzWZlsALNm9qvGjDOlVLjCUYHZOor9HLHADC8MVmg15XThd1lcaK
+EKFjhUIW1UH3LT2gh5qfhHL1ZHLCBIiwtqK+wdYFKnRl00inhryVLADf40xu3rZr
+TTX6oij5Z4QLKQOjs1Mipd8UXD5DABEBAAHCwXwEGAEKACYCGwwWIQR1f6t87RgU
+rhW0g25fsCdHQgNFTAUCai73FwUJHM8lFwAKCRBfsCdHQgNFTF/1D/9I1JVHn1Lk
+Bi1dhAUqx0OVm2LisrKuQ2wfUqWSge4tCuQHcZYE99Cgjg7WvTBZLMbotAxYdTRh
+ag3kqzRSy1cYz6SWXGn9/5Q1XH8L8y114i0NY4vxDJ52Dg2HU8NXCepsVqjl/rzj
+kQ4UbKNivTv10TrwOSSss8WCiF746thMHVlMadI06N4w9S9tpK0zfL56eR/haQJN
+IAHcYpEKgVHODWEGzW7Lflp5foHM/bzN0RhvgvuCS5hRdbHUKmibT1xu1jAr32rW
+x5B3rfM3t/GZJImPwZoTnhNrZRWt5Yq3KKWyXjg+BUWKeCzDpSWrMDF4AwcQ5/9d
+8J/TC66835CFYGIPQjSi7mR6OygFkzI3mQ1tqK4KBdrTcVpgDlR8Gz7BrCH/B8M5
+SdhdqTTTr8vQn2dMMGzKKt2JV0DCDb5M2ROwa/WcLcRNA2uqY7L8FTJvFqab0G0a
+k6a5HaiGrCsj5qbK02CgnWyn/FMwoD4Jc2oo/QUILukDLblXbBgk5RbwUwWaE2L3
+rWd1U7xwpASY/mYqzpt4UfbwmTKdAsgR07AyFhg3AxfzsbEp/VtYxDc8iG1JF8DU
+y00VGYkz676Ilgb+w3fKj0wp6FjVlK6ZOehNEYmLmMNbdKN6lINqkQXuOr6EOdxS
+0esX7WE9nMZr7c/TEVryZvr053B746BqCMLBfAQYAQoAJgIbDBYhBHV/q3ztGBSu
+FbSDbl+wJ0dCA0VMBQJoTb5aBQka7exaAAoJEF+wJ0dCA0VMdb0P/2bvqk6XPYyZ
+YJ6sglYUHrf4Gf1/plKfH1fSHRJ8kQe2M5bUoW3Qjn16+53DX62usUOSiRC/4ics
+YeZGqIgfpy18Ud2ZXYRBWEIMYJwiXFPeyFkuMSd5zOKmrS0g5V+g9w6O+N8U+1qb
+a/C/hv0Js32h8SspfLpyoF6BL6fmBCObl10bX4Ztwug4sqgWCirX0ewjPNbvGUyj
+A2M331xyIktGBiKRs5a+4PpZbumLBffwh/CVT3fbe7UwveGAtsdEHaJ/ZuTKV1ev
+s+IebBVm2YuTvTphBmhVTn4d64LVfTF9XmE1FUthx63i7+MY2fgnaGvjqxqj064B
+l5zh73rdEoTQ4npHMvpkiNDsRj/MtdMrTing1nHSi/FT/i+/Y0RjQNcO0ChJWKbD
+rci8AAVeFnYoxf6Vx5jCpQhdXUb341ej1yNDThU7ZOn3EIBVPsb4HZHtiHYH/ZFg
+hbDB8s6j+AhIXfcJkjju9gYr48XUdob6opyLjfOViaZFmCHvZsgjjrs2L1PcAioZ
+aiUNxXTtEjTxJFfWhLD4SmM0LwgmsZAzNR+Uz9fxooClHcHVikGXuObSwjYvzDTp
+vImeCuEv36fXNs2S4ww9qI9JcJT2xXPYKpjLXS/zmComeBdErrxl7F8mtMgBBzXI
+FvxX449A0Q6y4gXZxtFBUi9ejm9wR9YbwsF8BBgBCgAmAhsMFiEEdX+rfO0YFK4V
+tINuX7AnR0IDRUwFAmYexIgFCRi+8ogACgkQX7AnR0IDRUz9lhAAr/iOkETGJUFQ
+j6G+R61X6KbIoTWEmdxfBBSLLLpmmDL3vLRUAulHb/7b0uqrjLzWhWNHODrA7+dC
+3d1wT6/Ri2hbA/RJwssQf1ml2kNw0iTGNmj6q8TBLHCsbmbkSWpSz82/N8jjRij1
+qGvGp6SKDNMHc+BMqZxteCusxvhElBYWG2Kwal5BNu3mhhkPvx5H301Tm7wr6+CB
+iOE5pEenSK0GpL+9MN5LzDXAi0g7nZCFnowR5KLvaL7BDa6d9sWXHFvjixN4essX
+pFVBsO7jn/yCf2XrghJ7bx4P4h7W47B7vQq45ud58VO0WGJp71Jd8a4xIEm+NG2k
+L4L9XARN9awRpd2s1U0nnwoA4lzNw6842klyJ9saGqd34qsSAbSuJ8d+ooplDp+N
+fQ9b5q/QXEgs5trB8N9U9+ap9RtZ4mb8RtbPGU87kMRmcUs7cCmfc8o3CkYz3UHr
+7MKebRFzcGwelbAs9Z4GRaZV0s0V4NUZlojOiMEGPvRtSHmsUTQwRIlzBW2N2mv4
+X/J1cI3R6wAm/EHQ0sNjYeFPxDyGnDGjBdl14aHZ6mrKGtfXz1wOz5w6UZloeRVK
+G/PGS1N+VOqy4JJgNwFgDvnAx3WOIUfHjM2OAIFf/44x9iOt5uXq1nkHvclxAO6O
+VhrTW4kYI5kjObirH5d39zGz8xHTmcDCwXwEGAEKACYCGwwWIQR1f6t87RgUrhW0
+g25fsCdHQgNFTAUCZDsQuwUJFts+uwAKCRBfsCdHQgNFTCYAD/4vXGxq+1TEfCoD
+/6jSPctKqwONt1ZDY87yYjZkwoVDxTszH3BAdyfmn6pRW+OsD04dwYNo8yfym+6q
+TPss0Q2Wfh+RrHDoHJa1bEDNON6EsFV2aOKMfhnzR3jdL5hlr6bQhWaSUxmQaicU
+wxjrzCcWbJ43TOpIevn6rbORfcRt9DM6B7QVqrKrn4LEliYKlaV4QazVfT36GwFm
+HKdBe3VF2Vp+wAtLkciBLpKVdEndZD6zfzIsdwOI52O/IwhAx2pPDNao1xLK34e7
+L4KLseAkaIJkFQZWyMZkrzwL29ZFKBWbojeplePcsU6GBl1XpssYb/kc/ppZikaf
+3WVIkSxQ9mPrdRqr/cTwSn+8AlRUaT5FFMrHwPe0VOtL315GP2ZbYGi73cQFBJ/T
+FW7VjkQiF51ipjoF5A1v3BbTygwClqPC6efh7AkyxSeKN6c9vKe8m6bvprNkWyiD
+1ghFh3TiFCI86mGTxiGXKs+GxEJ2MLByhsFQ5b75pR4ls9VQGs4xe4sWWQlCid6W
+KfD6+Jc9KY7XnkchkbcwqfSLeVMGPB36N3xDKP04YUjeeN3VqS2uh6iHD3mExr7M
+Yo3Zd3dWUFttSdScbRfOxclCPHVlcTS3BuHpY145YlPlqDXxC705B+3DuCa7vAkn
+gkXk+rw1+fihAvNujvIvALFbQQUCxsLBfAQYAQoAJgIbDBYhBHV/q3ztGBSuFbSD
+bl+wJ0dCA0VMBQJgZKA9BQkTBM49AAoJEF+wJ0dCA0VMHHYQAKSs8J8WrdSijV6d
+gqUF5Nl+bFFjdUvxfCu30DM9NQ+I4nyY4FXG8NAT+nlyh7qZBaefFRZiEo1RkZko
+Uz/Qc0lPpqHCA5s5NFGXx0LL6odN8kOMq9QuLQb7Q/DAOBPB7L0S1rViHHaNThAg
+RwSGNfALVN6LGEVxpoRASjbwhzkl2raV1R86ZGiQWV8eP91TrcNCRMrhhfqwtz2T
+KFSpZ1EPMF4i/AHWDwT0VxiY4gISUiM9LX9gnlWbITZcblKGUnKGnf4JhKT5SDRz
+bhMwEOBNrPKIVI+iivA5+aBtc05Tc7RYYB2NFdNX66OvkIByL2JXXvqcAOICCKMx
+vtaGxXaZVhOdEnMNlKwJFxUahSTLl9e8ut3eUs182dQgMwwCfJ4UA0wDJPTwVBQH
+a+d1U9AFauoyGrO/YDKVyw+9FIZvUQ//czrl4v8igQoL16loLoeXScZwo2Py7ZGV
+C1WISrDZLgzCh+9nu+8hMuRuf1gTpFzMMwc3JXMHQw25CS7tFHNkZeJYFerIRhFB
+zF+evpZNQ2ToBMgp7uVV0dcAystWwMXq13ISuqNUjnkHJsts+Tm6k6yffAdLGPzw
+QaSFVnHfmfUjPE7NMeDvqgu58J5W6mVXVmMeKdq+ct1mLGcs+369hfcXx9UtJ6Ra
+LTLA6VdukTrMPblnMUajdRqPjui9wsF8BBgBCgAmAhsMFiEEdX+rfO0YFK4VtINu
+X7AnR0IDRUwFAl5RbGcFCRDxmmcACgkQX7AnR0IDRUyjmQ/7BRFGIFPOA38shl6S
+YfoX8IuHUTBVue5hevXYGDDp8EPgT4L9UCEfYvARr66X6h/zLNz0Osp4/1UhxXUu
+aR4REuNZ+O0rPBfaLnlyXHmGtGpqqFiOYpuKdpDRp7zneXhdL1cGWMG3+OLEHPd/
+geyPPYiC5M5RkcHguUEkYHhfM7fwqTwCD0qJWdUXF35md8ASTy2bWlk6QD1HzTBu
+658Y6hyj7UN2qXfcsJfgd9ke7X4d09PzysD2kC9nuZwgK2V54lOJWuBc2g/0YbCa
+qbgtOieG73bllWnjCj9A6aBJXel6E9fYE95BGkPmoYBsLR3vLRFp3DAqRIjBFqGN
+H6ctjDAlWGvkrBHLQXJdNH+uRz13KdfStFzlrVEBBcGL/pWUWG+FZOgv+3EMqqQZ
+ObkbWp/5gmWgNlDh94/aGRLRG6emnDZIIR5zwAODrbHTz/Cb78j9khHDnuupDWcq
+TEnUAm41NwPEFCF/jljLYlmg0ks7U0YUBp3GHfyoSC9zyQf7JE1j25gRE7w99VvZ
+hlXCfz7wm7t3r4wuuy4LL4Y56jlhcejXObZXmSaDIHFPPa6fAtKY4zJstIZcfq7w
+QvuHhP22MzBqJ+YiIu0DhngwD6b3Vf8Q8n7N1cbSyU34lXeUIw3BhVkbSvcrkjg8
+Xs/2yjEytbi5oycNGOxCFMioKKfCwXwEGAECACYCGwwWIQR1f6t87RgUrhW0g25f
+sCdHQgNFTAUCXHquxwUJDxrcxwAKCRBfsCdHQgNFTF1oEACOYbeTWQVf80J4KdV4
+03F+oT+zqMcUu/+FjZKNqoQrRG4GwfpeV17pyNGn5TWzXYQ4gJcnMCnI9zCdAgRn
+rwv/+yw1uaDfwSCN68r1AhXd1d58JhGfxfPSMT24ABkdKfYD3kdmMNCcwXlOhD/U
+v+il4JVvs0m/3za3r3/wxDYyOGDoyKfOW8MwfmWQe0ojadFZniqLFkQMunjnskeA
+1mzObdJQzsSlQRKMe5Qzmrdy+JIOKJdaZMP44sgF4M8Ak6ra3nZcRXHKozNkKfa1
+gfj3A8vScEV8M/p0nKGQ1Qd3dMVmlhGQieFbmZLm+pki98C1uBW27WFytQro8gwZ
+b43N4Hazv8VWCHtzzaVv0AqLZoQkYOErQ5kN34bJ4bZfWo7heSVuR/i+fKLsw+Jc
+B5+yPKAuY+WmKsPYqPtuVqTqGZppFxoN9ypKpvcvjj315XEVaYbpQPOfVjk49Oxm
+378FJtYiNPV+CPPUEWNHEalpuja+zIAa8lF4Dc4mHRrZN2nOoTYZvSy6IJOQVr10
+F7lNPVoJcFs8l6n5ib+I/g3170P4gKCo1HqEs+rYHq082q51uhjYh7/sRVw+kpNW
+VhQ0mCQiqCjLun60YnkC/jwD6DHsQd9I4cZ59A5EBzMOYOnb7drtGjiC6DFIuuuh
+57HfX5uzklFeTKezRweMQVF9U8LBfAQYAQIAJgIbDBYhBHV/q3ztGBSuFbSDbl+w
+J0dCA0VMBQJamvAbBQkNOx4bAAoJEF+wJ0dCA0VMGtUP/RGdD2uQuL5ZywTQpg/6
+9/3Jr8OMySUgRaWunNATfGyc57L5HIV8R/v25fsQi1bsTSwWdHo8REuhiXFyO7Be
+k8ziax7Ia/j4PkJBWjlfiKKay/nJeGpoibWN7pP34t8DVsQM/WeEAWLd+kCG3OgQ
+9M5ep+q+U1hxNxoh3a2pDvuPeWeY59ubB6Ug7jH+yI4V+7t6oUVAIbmHnGWyPasI
+JrYnx7u+KX8h9KBgs+LXdVi2db5spw7FWNw4qlALZg1ixdSRQjkLVBKKBjFS8WyV
+rTNlADLMZT3KNHxQEHf9vf+dYRQ8fCgUo5+HqrARTSAv0x7DTUzzipbgMVlUCSvP
+1vOqnmBPbxplXKiSZ/IPOiG9KFNJT17oT3cPErNboChUiq+50GFt3H0p0tMYxDcP
+ZlFemE6Pcn2pEEJ1QLZLNdVzxqIWCSI3BdAOE3fyeBDk5eyUdl49jFRzf+Wp65Tx
+QJ9RMogGJCFcFJvoJCa0X4Ql7Ne/VckxjNZ67C0A3+y49LSJ/mgKzS4el8u9sJd4
+2t62U1OFqMLlW5uFJLaoKcWHzr4NwUmudr1HCoChYlJv6G+0vgE1I8b6Xno4i6Ig
+Slx2eTqJJvI47Iu19G251LdL+6oU+vWzjSsdDpwpnQul0zY/CtL9E0VTymAXujKz
+V2fQC5TzN+5O9PVp7xJS9MfuwsFlBBgBAgAPAhsMBQJW2HvGBQkLWd1DAAoJEF+w
+J0dCA0VMtOkP/3vhFlDk2J0XwKwwpO/2oQccab3DInlllikvUbt1frAdcUsTbzMY
+Y6Xczu6Pip038cIRGrUD3FDCcfC1hUxETMd9Wr/u5epOadkzDniLso8P+ZJkHy+G
+6a4QJnzkOp/Gn0Eqb90OvZG6daCkqMkoX0Rei8cviF5txEWxfOptaY2R8oDR7+PX
+cnj0VnU3DTH09F2+Y2ExA9OuwHNEshKFJoWBqsUs4r3TsYjl8809uZFnqcunE8zM
+DVxkxDgIn0GrqEtVykNj4D67AQCFBPCLNj/+Qf4r4RP8KnxjxXw2qOzMln7M7b71
+01n+fUqAOmbwxHSzbJ4EqalqxLGoC2STJZy3Ke+ID67Wgpyro/fB5WkLKSZ+6+Ic
+yS5s5xc62m/rr4InGO1Wz82n4vGW8uqKZ0eWp4T7Emvip9G1hyrisX4Aft/QuIrK
+cw3haoFXzAcyNKsr8XZwaJhnA2hJ9drQ1LsUKQUAeLRlVFSWXgCT2RxyDDyKOq/V
+ZpFbwP+KReLV+BAKlklcvpEZTYMJQxteZJKVkyR0HVFnZXL4sfrLTVTnPs+jUVx/
+7xza+FFMm28Q6nMNSyGBIydm/9cLcmv3VZzhrjVUD1pF2j3WAYmLOCKPreikygOS
+va0VGFLPUXFH/yM0avP8nXSby8MFM7NUaOeUBaKfoHmeypGXTvhTWScMwsFlBBgB
+AgAPAhsMBQJU9zR1BQkHl2JyAAoJEF+wJ0dCA0VMMV8P/jvYlb23Jw8kqnfjs1+x
+CYJk1JPk49ODiTWeEjfMDspAwUsd05Mxp5sgwEcDIc8X1Kzu6yf752hkMsZKDFWV
+s9weTKdH/IPP2PfzErsBUJlg+dX+Hyokx7mA8+BLVXwad8/60461DeLjiF89oshE
+9jhtTAmZFPzKQpcbRzjK4iJ37680fhTYKsFQbpkRKACLBD3fkumUubj5TFznsC8i
+GNme1f+lUSeqwEEHWdE9u8HbZegUPBz788jnZbcoJJyWWXT2z3g6xQDqklK1s8uO
+YOeH1s0IiwFwqwHLUiFc1ZnIxDu+0I7zag934AzkQDUWXW+4VyD6bPFt/toVwGc0
+2jAtPth50gNbEowj+l4Zj+k7pIFnohDXd4Rd5rGWK/MgezdyDjtPMxuUe8KzJ6MI
+q7eOvFLkf0zewXhxcb/DqFb5/YV+oYw18hzVybnJA8ZEodc+ZV0alQ2tBLIcr/u2
+O2wkTUFhVkkGZEx2/vVvWizg4DTVwW2+7N4S189dL0rOyxEjNmWywtFwpte55hpK
+nj/YMdMY/n60HoM9j/CV9G7uxM3AncdL3/bnItcTbKRU227WmweIkEyXUa5MAw6Z
+ziZA13JYdWAuF7NufirkSsVCHx4WFkzlxPHSs8FFoYw+FeVD/lf7Qo/GNbjYYljX
+VNWHi8oAh1L0D7jmrwvYz2HOwsFlBBgBAgAPAhsMBQJTE3I6BQkFs6A2AAoJEF+w
+J0dCA0VMCgMQAJiDYorgJD61AmUicq0+QiMpGxDvvsMkRt/T7Zw3xqmXHqz2t0g9
+wrvV5GFo1Eeu0ZBzAXkjLDjWbeo7uuYKwxmvWkXYuZwjwl3uyLshXcKdc7HPSRXl
+hFxVBLQFJjEiwRFtcwLsrYJCV1SWUybNU2rNRfpNZ1eAYGfHDA/yH0ow0oktZcDH
+DIim8tIWcHF+ZPgqZJnooEl+Sq37eMCar52MGlGJ6vSRbo9lZ0BKovb6zRFdgEDo
+TVjHLavAKRaqdP9rKagjgmieWeykik0aKF3wDjM63yvHQhg5rJXZ5tAtbPnSE45z
+fX6sU1k1rQYlsKjA0wGI5AcRWsWoKhp4++35qNQVa5qNhG2nGyNrq0qfmh8fWTWz
+RoLAPzeIWvhXh+kY0cZraBs2vqWvcZN/1r2XkdNTGd7C61l5vUwW40RSP/5gMgC+
+z7vHXCvND032KmRp/8/hbv8NgoTeTnt7qRtWHHQ0UjEDND7064PsR7VO3EYLwDEj
+BWd3aIA/mWmKcsHhxjKXDI2Ea05r4EVH+53fj+c9TrNRKl3eXDdpqZZk2Za56ON+
+GBolVNpouU1pnCLeIXnamNuEFW8inv8kTKsMk/Tk8Jhktw9pTG4B+NJ4FACzPhrU
+G/3pHMSW/0/vE8RCnSgw85Xr8L/rsMRHDWi+d6qdJftnk+NZGtfdlirrwsFlBBgB
+AgAPAhsMBQJRMRWFBQkD0UODAAoJEF+wJ0dCA0VMkh8QANup+aYJHil1Wa6/XlBy
+ZpBUXTDz9Wf1BtCppiSKlvS+IXS04VP8RMcYehp9/JAWHHsmxWr9go3WIocXT3la
+DpWXikbWHc0sDs6UAgrovP4Ua2NZHWDPuX339U9V1GSk+kR7ucSQXkO4mqeVfncw
+C86YUYcKEzdT7qwzjFmZ9PBw+Oko+iCAoqxpy6+vmOWBtAtZX2P0hKh8DxAKy6nb
+tzW7kcXOGucG33804k2GBw80TjKsk/vw/ALNX0/TKcOoi3zbFEtwp5mGHAiIASX0
+Sxrs4uABOUuaC7t2CoCkCuBnjrAAs9AOzGZN6XY3kUNm/xIS3nJAE27GtkFET8a3
+kiMRI9SFvFe45c+RAJ7tm0iXrvCEvBaRvE+iQJy00ju0Va06BdTZaSjMkPfznU4w
+n9VpS/SsvfNIU4NA622/qEwTr5rKkau5YQ7R1gpd1h4MAAHx+yAJ4ymPL4Quk4Ul
+XCUmgqtdFVoZA8jPypQCF3Mc0y6LCSOu5ApIge/B2WAuTCwTrD3WvidpYS4RE2AV
++By4+ECK6j7fJVRPvmG9px7yQgpbPZV+4n3wp4unHi+UN7jG7nIBU8JB9k5SE2A6
+bO/6P3nWIi1IY8rOxtY9eMTwCTDuOc0f5LBK4cYVMGLGGjPViX966TMRgswoC3ZT
+mQ8rpz6C+7RBoRBPv1iSLYDhwsFlBBgBAgAPBQJPQQWAAhsMBQkB4TOAAAoJEF+w
+J0dCA0VMed0P/17S8Gvt2gfYAY9HVt70Xv5j1+c4E2R4INUeT67Y8ay5OC3PLKEj
+eveRZO/y7/xTRhfa0/HQ9HE92y8/ci5J3HUIJWrY4o0Q3B7yFlRd1suUMrCRZYGc
+b9DwOOONnMhuwy/ndyGpYK9+8VudQ/FMSFcO+IVLAZi/3DyEQQrSHGSmfbJvRBPl
+hDkQggoy84h6UacZ7H5b5mD5MpoJbII94uC8uXRrh5j+wOHS80IfqSmeC0jTm/H+
+S4AINsbFtJ2K0nw3sREHGo5QoLtXPhn41lnU7PQST9egWxhduz5rWsVK1V/0ipW7
+usLr1ngamTUp7IGtTFhQN8Pm7naQbQ4F+ztAmr5pzxpGo1j5bI9oOrbxP6gNbLzd
+76pqEjdSjsX1YqouLpwcSGv8bRyr2wNk4HZWC61PXPVwC2IZGo6P8u3fIHMg1rnH
+27qrVXre7zu0ip9xY78kdal0si83b1tF8BbOSuwWQap5ULGG949qWeqlEbN/Ki3G
+ToXo1A6+My3R+WgVl0BCSi3aV37/XEuV6KXz5kYE+fFnqlordqqHu8rGeyNtmS7e
+wU0t/v2grE+K1ybmAr97auYp76qze8AF9bqacxroJH4pB2O0F/seTGlmDf3QvH98
+CZD1VnjKOim758GAH8t3godbPQ/ADmbEQr0bfDjHIahp7aU3J/nSwf5q
+=pcdu
 -----END PGP PUBLIC KEY BLOCK-----
diff --git a/b43-fwcutter.spec b/b43-fwcutter.spec
index 7485b46..8797b7b 100644
--- a/b43-fwcutter.spec
+++ b/b43-fwcutter.spec
@@ -6,12 +6,14 @@ License:        BSD-2-Clause
 URL:            https://bues.ch/b43/fwcutter/
 Source0:        https://bues.ch/b43/fwcutter/%{name}-%{version}.tar.xz
 Source1:        https://bues.ch/b43/fwcutter/%{name}-%{version}.tar.xz.asc
-# gpg --no-default-keyring --keyring ./keyring.gpg --keyserver keyserver.ubuntu.com --recv-key 757FAB7CED1814AE15B4836E5FB027474203454C
-# gpg --no-default-keyring --keyring ./keyring.gpg  --output 757FAB7CED1814AE15B4836E5FB027474203454C.gpg --export --armor
+# Refreshed from keyservers/WKD (same keys, newer signatures):
+#   sq --home=none network search --output - 0392335A78083894A4301C43236E8A58C6DB4512 69100FFB90E87A320DF8643CBEC39009E8321A61 757FAB7CED1814AE15B4836E5FB027474203454C | sq keyring filter --experimental --cert 0392335A78083894A4301C43236E8A58C6DB4512 --cert 69100FFB90E87A320DF8643CBEC39009E8321A61 --cert 757FAB7CED1814AE15B4836E5FB027474203454C --output update.pgp
+#   sq keyring merge --output 757FAB7CED1814AE15B4836E5FB027474203454C.gpg.new 757FAB7CED1814AE15B4836E5FB027474203454C.gpg update.pgp
+#   mv 757FAB7CED1814AE15B4836E5FB027474203454C.gpg.new 757FAB7CED1814AE15B4836E5FB027474203454C.gpg
 Source2:        757FAB7CED1814AE15B4836E5FB027474203454C.gpg
 Source100:      README.too
 BuildRequires:  gcc
-BuildRequires:  gnupg2
+BuildRequires:  openpgpverify
 BuildRequires:  make
 
 %description
@@ -22,7 +24,7 @@ See the README.too file shipped in the package's documentation for
 instructions on using this tool.
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %autosetup -p2
 
 cp %{SOURCE100} .
-- 
2.56.0

